2015-05-25 21:03:19,116 [main] INFO  co.solinx.forest.container.spring.SpringContainer (SpringContainer.java:22) - SpringContainer start
2015-05-25 21:03:19,752 [main] INFO  org.springframework.context.support.AbstractApplicationContext (AbstractApplicationContext.java:510) - Refreshing org.springframework.context.support.ClassPathXmlApplicationContext@7fbe847c: startup date [Mon May 25 21:03:19 CST 2015]; root of context hierarchy
2015-05-25 21:03:19,851 [main] INFO  org.springframework.beans.factory.xml.XmlBeanDefinitionReader (XmlBeanDefinitionReader.java:317) - Loading XML bean definitions from file [E:\forest\forest-demo\forest-demo-provider\target\classes\META-INF\spring\forest-provider.xml]
2015-05-25 21:03:20,590 [main] INFO  co.solinx.forest.config.spring.ServiceBean (ServiceBean.java:32) - ServiceBean 
2015-05-25 21:03:20,631 [main] INFO  co.solinx.forest.config.spring.ServiceBean (ServiceBean.java:32) - ServiceBean 
2015-05-25 21:03:20,650 [main] INFO  co.solinx.forest.config.spring.ServiceBean (ServiceBean.java:51) - _________________________org.springframework.context.event.ContextRefreshedEvent[source=org.springframework.context.support.ClassPathXmlApplicationContext@7fbe847c: startup date [Mon May 25 21:03:19 CST 2015]; root of context hierarchy]
2015-05-25 21:03:20,650 [main] INFO  co.solinx.forest.config.ServiceConfig (ServiceConfig.java:40) - Export£ºco.solinx.forest.demo.api.IHelloForestService
2015-05-25 21:03:21,359 [main] INFO  org.apache.curator.framework.imps.CuratorFrameworkImpl (CuratorFrameworkImpl.java:224) - Starting
2015-05-25 21:03:41,484 [main] INFO  org.apache.zookeeper.Environment (Environment.java:100) - Client environment:zookeeper.version=3.4.6-1569965, built on 02/20/2014 09:09 GMT
2015-05-25 21:03:41,485 [main] INFO  org.apache.zookeeper.Environment (Environment.java:100) - Client environment:host.name=linx
2015-05-25 21:03:41,486 [main] INFO  org.apache.zookeeper.Environment (Environment.java:100) - Client environment:java.version=1.8.0_40
2015-05-25 21:03:41,486 [main] INFO  org.apache.zookeeper.Environment (Environment.java:100) - Client environment:java.vendor=Oracle Corporation
2015-05-25 21:03:41,487 [main] INFO  org.apache.zookeeper.Environment (Environment.java:100) - Client environment:java.home=D:\Java\jdk1.8.0_40\jre
2015-05-25 21:03:41,488 [main] INFO  org.apache.zookeeper.Environment (Environment.java:100) - Client environment:java.class.path=D:\Java\jdk1.8.0_40\jre\lib\charsets.jar;D:\Java\jdk1.8.0_40\jre\lib\deploy.jar;D:\Java\jdk1.8.0_40\jre\lib\javaws.jar;D:\Java\jdk1.8.0_40\jre\lib\jce.jar;D:\Java\jdk1.8.0_40\jre\lib\jfr.jar;D:\Java\jdk1.8.0_40\jre\lib\jfxswt.jar;D:\Java\jdk1.8.0_40\jre\lib\jsse.jar;D:\Java\jdk1.8.0_40\jre\lib\management-agent.jar;D:\Java\jdk1.8.0_40\jre\lib\plugin.jar;D:\Java\jdk1.8.0_40\jre\lib\resources.jar;D:\Java\jdk1.8.0_40\jre\lib\rt.jar;D:\Java\jdk1.8.0_40\jre\lib\ext\access-bridge-64.jar;D:\Java\jdk1.8.0_40\jre\lib\ext\cldrdata.jar;D:\Java\jdk1.8.0_40\jre\lib\ext\dnsns.jar;D:\Java\jdk1.8.0_40\jre\lib\ext\jaccess.jar;D:\Java\jdk1.8.0_40\jre\lib\ext\jfxrt.jar;D:\Java\jdk1.8.0_40\jre\lib\ext\localedata.jar;D:\Java\jdk1.8.0_40\jre\lib\ext\nashorn.jar;D:\Java\jdk1.8.0_40\jre\lib\ext\sunec.jar;D:\Java\jdk1.8.0_40\jre\lib\ext\sunjce_provider.jar;D:\Java\jdk1.8.0_40\jre\lib\ext\sunmscapi.jar;D:\Java\jdk1.8.0_40\jre\lib\ext\sunpkcs11.jar;D:\Java\jdk1.8.0_40\jre\lib\ext\zipfs.jar;E:\forest\forest-demo\forest-demo-provider\target\classes;D:\apache-maven-3.1.0\repository\log4j\log4j\1.2.17\log4j-1.2.17.jar;D:\apache-maven-3.1.0\repository\io\netty\netty\3.7.0.Final\netty-3.7.0.Final.jar;D:\apache-maven-3.1.0\repository\org\springframework\spring-beans\4.1.5.RELEASE\spring-beans-4.1.5.RELEASE.jar;D:\apache-maven-3.1.0\repository\org\springframework\spring-core\4.1.5.RELEASE\spring-core-4.1.5.RELEASE.jar;D:\apache-maven-3.1.0\repository\commons-logging\commons-logging\1.2\commons-logging-1.2.jar;D:\apache-maven-3.1.0\repository\org\springframework\spring-context\4.1.5.RELEASE\spring-context-4.1.5.RELEASE.jar;D:\apache-maven-3.1.0\repository\org\springframework\spring-aop\4.1.5.RELEASE\spring-aop-4.1.5.RELEASE.jar;D:\apache-maven-3.1.0\repository\aopalliance\aopalliance\1.0\aopalliance-1.0.jar;D:\apache-maven-3.1.0\repository\org\springframework\spring-expression\4.1.5.RELEASE\spring-expression-4.1.5.RELEASE.jar;D:\apache-maven-3.1.0\repository\org\apache\zookeeper\zookeeper\3.4.6\zookeeper-3.4.6.jar;D:\apache-maven-3.1.0\repository\org\slf4j\slf4j-api\1.6.1\slf4j-api-1.6.1.jar;D:\apache-maven-3.1.0\repository\org\slf4j\slf4j-log4j12\1.6.1\slf4j-log4j12-1.6.1.jar;D:\apache-maven-3.1.0\repository\jline\jline\0.9.94\jline-0.9.94.jar;D:\apache-maven-3.1.0\repository\junit\junit\3.8.1\junit-3.8.1.jar;D:\apache-maven-3.1.0\repository\org\apache\curator\curator-framework\2.7.1\curator-framework-2.7.1.jar;D:\apache-maven-3.1.0\repository\org\apache\curator\curator-client\2.7.1\curator-client-2.7.1.jar;D:\apache-maven-3.1.0\repository\com\google\guava\guava\16.0.1\guava-16.0.1.jar;E:\forest\forest-common\target\classes;D:\apache-maven-3.1.0\repository\org\javassist\javassist\3.15.0-GA\javassist-3.15.0-GA.jar;E:\forest\forest-config\forest-config-api\target\classes;E:\forest\forest-rpc\forest-rpc-default\target\classes;E:\forest\forest-rpc\forest-rpc-api\target\classes;E:\forest\forest-rpc\forest-rpc-jdk\target\classes;E:\forest\forest-cluster\target\classes;E:\forest\forest-config\forest-config-spring\target\classes;E:\forest\forest-container\forest-container-api\target\classes;E:\forest\forest-container\forest-container-log4j\target\classes;E:\forest\forest-container\forest-container-spring\target\classes;E:\forest\forest-registry\forest-registry-zookeeper\target\classes;E:\forest\forest-registry\forest-registry-api\target\classes;E:\forest\forest-remote\forest-remote-netty\target\classes;D:\apache-maven-3.1.0\repository\io\netty\netty-all\4.0.27.Final\netty-all-4.0.27.Final.jar;E:\forest\forest-remote\forest-remote-api\target\classes;E:\forest\forest-remote\forest-remote-zookeeper\target\classes;D:\JetBrains\IntelliJ IDEA 14.1\lib\idea_rt.jar
2015-05-25 21:03:41,490 [main] INFO  org.apache.zookeeper.Environment (Environment.java:100) - Client environment:java.library.path=D:\Java\jdk1.8.0_40\bin;C:\Windows\Sun\Java\bin;C:\Windows\system32;C:\Windows;C:\ProgramData\Oracle\Java\javapath;C:\Windows\system32;C:\Windows;C:\Windows\System32\Wbem;C:\Windows\System32\WindowsPowerShell\v1.0\;D:\Java\jdk1.8.0_40\bin;D:\Java\jdk1.8.0_40\jre\bin;D:\Git\bin;D:\Go\bin;;D:\Boot2Docker for Windows;.
2015-05-25 21:03:41,499 [main] INFO  org.apache.zookeeper.Environment (Environment.java:100) - Client environment:java.io.tmpdir=C:\Users\öÎ\AppData\Local\Temp\
2015-05-25 21:03:41,502 [main] INFO  org.apache.zookeeper.Environment (Environment.java:100) - Client environment:java.compiler=<NA>
2015-05-25 21:03:41,503 [main] INFO  org.apache.zookeeper.Environment (Environment.java:100) - Client environment:os.name=Windows 8.1
2015-05-25 21:03:41,503 [main] INFO  org.apache.zookeeper.Environment (Environment.java:100) - Client environment:os.arch=amd64
2015-05-25 21:03:41,504 [main] INFO  org.apache.zookeeper.Environment (Environment.java:100) - Client environment:os.version=6.3
2015-05-25 21:03:41,504 [main] INFO  org.apache.zookeeper.Environment (Environment.java:100) - Client environment:user.name=öÎ
2015-05-25 21:03:41,504 [main] INFO  org.apache.zookeeper.Environment (Environment.java:100) - Client environment:user.home=C:\Users\öÎ
2015-05-25 21:03:41,505 [main] INFO  org.apache.zookeeper.Environment (Environment.java:100) - Client environment:user.dir=E:\forest
2015-05-25 21:03:41,507 [main] INFO  org.apache.zookeeper.ZooKeeper (ZooKeeper.java:438) - Initiating client connection, connectString=192.168.140.142:2181 sessionTimeout=60000 watcher=org.apache.curator.ConnectionState@1a38c59b
2015-05-25 21:03:41,691 [main-SendThread(192.168.140.142:2181)] INFO  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:975) - Opening socket connection to server 192.168.140.142/192.168.140.142:2181. Will not attempt to authenticate using SASL (unknown error)
2015-05-25 21:03:41,744 [main] INFO  co.solinx.remote.zookeeper.ZookeeperClient (ZookeeperClient.java:42) - STARTED
2015-05-25 21:03:41,745 [main] INFO  co.solinx.forest.config.ServiceConfig (ServiceConfig.java:45) - interfaceName_______________co.solinx.forest.demo.api.IHelloForestService
2015-05-25 21:03:56,841 [main] ERROR org.apache.curator.ConnectionState (ConnectionState.java:200) - Connection timed out for connection string (192.168.140.142:2181) and timeout (15000) / elapsed (35468)
org.apache.curator.CuratorConnectionLossException: KeeperErrorCode = ConnectionLoss
	at org.apache.curator.ConnectionState.checkTimeouts(ConnectionState.java:197)
	at org.apache.curator.ConnectionState.getZooKeeper(ConnectionState.java:87)
	at org.apache.curator.CuratorZookeeperClient.getZooKeeper(CuratorZookeeperClient.java:115)
	at org.apache.curator.framework.imps.CuratorFrameworkImpl.getZooKeeper(CuratorFrameworkImpl.java:477)
	at org.apache.curator.framework.imps.ExistsBuilderImpl$2.call(ExistsBuilderImpl.java:172)
	at org.apache.curator.framework.imps.ExistsBuilderImpl$2.call(ExistsBuilderImpl.java:161)
	at org.apache.curator.RetryLoop.callWithRetry(RetryLoop.java:107)
	at org.apache.curator.framework.imps.ExistsBuilderImpl.pathInForeground(ExistsBuilderImpl.java:158)
	at org.apache.curator.framework.imps.ExistsBuilderImpl.forPath(ExistsBuilderImpl.java:148)
	at org.apache.curator.framework.imps.ExistsBuilderImpl.forPath(ExistsBuilderImpl.java:36)
	at co.solinx.remote.zookeeper.ZookeeperClient.checkExists(ZookeeperClient.java:135)
	at co.solinx.remote.zookeeper.ZookeeperClient.createNote(ZookeeperClient.java:72)
	at co.solinx.forest.registry.zookeeper.ZookeeperRegistry.createRoot(ZookeeperRegistry.java:68)
	at co.solinx.forest.registry.zookeeper.ZookeeperRegistry.registryService(ZookeeperRegistry.java:136)
	at co.solinx.forest.config.ServiceConfig.Export(ServiceConfig.java:54)
	at co.solinx.forest.config.spring.ServiceBean.onApplicationEvent(ServiceBean.java:52)
	at org.springframework.context.event.SimpleApplicationEventMulticaster.invokeListener(SimpleApplicationEventMulticaster.java:151)
	at org.springframework.context.event.SimpleApplicationEventMulticaster.multicastEvent(SimpleApplicationEventMulticaster.java:128)
	at org.springframework.context.support.AbstractApplicationContext.publishEvent(AbstractApplicationContext.java:331)
	at org.springframework.context.support.AbstractApplicationContext.finishRefresh(AbstractApplicationContext.java:773)
	at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:483)
	at org.springframework.context.support.ClassPathXmlApplicationContext.<init>(ClassPathXmlApplicationContext.java:139)
	at org.springframework.context.support.ClassPathXmlApplicationContext.<init>(ClassPathXmlApplicationContext.java:83)
	at co.solinx.forest.container.spring.SpringContainer.start(SpringContainer.java:23)
	at co.solinx.forest.container.Main.main(Main.java:29)
	at provider.main(provider.java:8)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at com.intellij.rt.execution.application.AppMain.main(AppMain.java:140)
2015-05-25 21:04:02,694 [main-SendThread(192.168.140.142:2181)] WARN  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:1102) - Session 0x0 for server null, unexpected error, closing socket connection and attempting reconnect
java.net.ConnectException: Connection timed out: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(Unknown Source)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:361)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1081)
2015-05-25 21:04:03,797 [main-SendThread(192.168.140.142:2181)] INFO  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:975) - Opening socket connection to server 192.168.140.142/192.168.140.142:2181. Will not attempt to authenticate using SASL (unknown error)
2015-05-25 21:04:12,878 [main] ERROR org.apache.curator.ConnectionState (ConnectionState.java:200) - Connection timed out for connection string (192.168.140.142:2181) and timeout (15000) / elapsed (51516)
org.apache.curator.CuratorConnectionLossException: KeeperErrorCode = ConnectionLoss
	at org.apache.curator.ConnectionState.checkTimeouts(ConnectionState.java:197)
	at org.apache.curator.ConnectionState.getZooKeeper(ConnectionState.java:87)
	at org.apache.curator.CuratorZookeeperClient.getZooKeeper(CuratorZookeeperClient.java:115)
	at org.apache.curator.framework.imps.CuratorFrameworkImpl.getZooKeeper(CuratorFrameworkImpl.java:477)
	at org.apache.curator.framework.imps.ExistsBuilderImpl$2.call(ExistsBuilderImpl.java:172)
	at org.apache.curator.framework.imps.ExistsBuilderImpl$2.call(ExistsBuilderImpl.java:161)
	at org.apache.curator.RetryLoop.callWithRetry(RetryLoop.java:107)
	at org.apache.curator.framework.imps.ExistsBuilderImpl.pathInForeground(ExistsBuilderImpl.java:158)
	at org.apache.curator.framework.imps.ExistsBuilderImpl.forPath(ExistsBuilderImpl.java:148)
	at org.apache.curator.framework.imps.ExistsBuilderImpl.forPath(ExistsBuilderImpl.java:36)
	at co.solinx.remote.zookeeper.ZookeeperClient.checkExists(ZookeeperClient.java:135)
	at co.solinx.remote.zookeeper.ZookeeperClient.createNote(ZookeeperClient.java:72)
	at co.solinx.forest.registry.zookeeper.ZookeeperRegistry.createRoot(ZookeeperRegistry.java:68)
	at co.solinx.forest.registry.zookeeper.ZookeeperRegistry.registryService(ZookeeperRegistry.java:136)
	at co.solinx.forest.config.ServiceConfig.Export(ServiceConfig.java:54)
	at co.solinx.forest.config.spring.ServiceBean.onApplicationEvent(ServiceBean.java:52)
	at org.springframework.context.event.SimpleApplicationEventMulticaster.invokeListener(SimpleApplicationEventMulticaster.java:151)
	at org.springframework.context.event.SimpleApplicationEventMulticaster.multicastEvent(SimpleApplicationEventMulticaster.java:128)
	at org.springframework.context.support.AbstractApplicationContext.publishEvent(AbstractApplicationContext.java:331)
	at org.springframework.context.support.AbstractApplicationContext.finishRefresh(AbstractApplicationContext.java:773)
	at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:483)
	at org.springframework.context.support.ClassPathXmlApplicationContext.<init>(ClassPathXmlApplicationContext.java:139)
	at org.springframework.context.support.ClassPathXmlApplicationContext.<init>(ClassPathXmlApplicationContext.java:83)
	at co.solinx.forest.container.spring.SpringContainer.start(SpringContainer.java:23)
	at co.solinx.forest.container.Main.main(Main.java:29)
	at provider.main(provider.java:8)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at com.intellij.rt.execution.application.AppMain.main(AppMain.java:140)
2015-05-25 21:04:24,795 [main-SendThread(192.168.140.142:2181)] WARN  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:1102) - Session 0x0 for server null, unexpected error, closing socket connection and attempting reconnect
java.net.ConnectException: Connection timed out: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(Unknown Source)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:361)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1081)
2015-05-25 21:04:25,896 [main-SendThread(192.168.140.142:2181)] INFO  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:975) - Opening socket connection to server 192.168.140.142/192.168.140.142:2181. Will not attempt to authenticate using SASL (unknown error)
2015-05-25 21:04:30,896 [main] WARN  org.apache.curator.ConnectionState (ConnectionState.java:191) - Connection attempt unsuccessful after 69534 (greater than max timeout of 60000). Resetting connection and trying again with a new connection.
2015-05-25 21:04:47,000 [main] INFO  org.apache.zookeeper.ZooKeeper (ZooKeeper.java:684) - Session: 0x0 closed
2015-05-25 21:04:47,000 [main-EventThread] INFO  org.apache.zookeeper.ClientCnxn$EventThread (ClientCnxn.java:512) - EventThread shut down
2015-05-25 21:04:47,000 [main] INFO  org.apache.zookeeper.ZooKeeper (ZooKeeper.java:438) - Initiating client connection, connectString=192.168.140.142:2181 sessionTimeout=60000 watcher=org.apache.curator.ConnectionState@1a38c59b
2015-05-25 21:04:47,003 [main-SendThread(192.168.140.142:2181)] INFO  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:975) - Opening socket connection to server 192.168.140.142/192.168.140.142:2181. Will not attempt to authenticate using SASL (unknown error)
2015-05-25 21:05:08,003 [main-SendThread(192.168.140.142:2181)] WARN  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:1102) - Session 0x0 for server null, unexpected error, closing socket connection and attempting reconnect
java.net.ConnectException: Connection timed out: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(Unknown Source)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:361)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1081)
2015-05-25 21:05:09,104 [main-SendThread(192.168.140.142:2181)] INFO  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:975) - Opening socket connection to server 192.168.140.142/192.168.140.142:2181. Will not attempt to authenticate using SASL (unknown error)
2015-05-25 21:05:24,118 [main] ERROR org.apache.curator.ConnectionState (ConnectionState.java:200) - Connection timed out for connection string (192.168.140.142:2181) and timeout (15000) / elapsed (53222)
org.apache.curator.CuratorConnectionLossException: KeeperErrorCode = ConnectionLoss
	at org.apache.curator.ConnectionState.checkTimeouts(ConnectionState.java:197)
	at org.apache.curator.ConnectionState.getZooKeeper(ConnectionState.java:87)
	at org.apache.curator.CuratorZookeeperClient.getZooKeeper(CuratorZookeeperClient.java:115)
	at org.apache.curator.framework.imps.CuratorFrameworkImpl.getZooKeeper(CuratorFrameworkImpl.java:477)
	at org.apache.curator.framework.imps.ExistsBuilderImpl$2.call(ExistsBuilderImpl.java:172)
	at org.apache.curator.framework.imps.ExistsBuilderImpl$2.call(ExistsBuilderImpl.java:161)
	at org.apache.curator.RetryLoop.callWithRetry(RetryLoop.java:107)
	at org.apache.curator.framework.imps.ExistsBuilderImpl.pathInForeground(ExistsBuilderImpl.java:158)
	at org.apache.curator.framework.imps.ExistsBuilderImpl.forPath(ExistsBuilderImpl.java:148)
	at org.apache.curator.framework.imps.ExistsBuilderImpl.forPath(ExistsBuilderImpl.java:36)
	at co.solinx.remote.zookeeper.ZookeeperClient.checkExists(ZookeeperClient.java:135)
	at co.solinx.remote.zookeeper.ZookeeperClient.createNote(ZookeeperClient.java:72)
	at co.solinx.forest.registry.zookeeper.ZookeeperRegistry.createRoot(ZookeeperRegistry.java:68)
	at co.solinx.forest.registry.zookeeper.ZookeeperRegistry.registryService(ZookeeperRegistry.java:136)
	at co.solinx.forest.config.ServiceConfig.Export(ServiceConfig.java:54)
	at co.solinx.forest.config.spring.ServiceBean.onApplicationEvent(ServiceBean.java:52)
	at org.springframework.context.event.SimpleApplicationEventMulticaster.invokeListener(SimpleApplicationEventMulticaster.java:151)
	at org.springframework.context.event.SimpleApplicationEventMulticaster.multicastEvent(SimpleApplicationEventMulticaster.java:128)
	at org.springframework.context.support.AbstractApplicationContext.publishEvent(AbstractApplicationContext.java:331)
	at org.springframework.context.support.AbstractApplicationContext.finishRefresh(AbstractApplicationContext.java:773)
	at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:483)
	at org.springframework.context.support.ClassPathXmlApplicationContext.<init>(ClassPathXmlApplicationContext.java:139)
	at org.springframework.context.support.ClassPathXmlApplicationContext.<init>(ClassPathXmlApplicationContext.java:83)
	at co.solinx.forest.container.spring.SpringContainer.start(SpringContainer.java:23)
	at co.solinx.forest.container.Main.main(Main.java:29)
	at provider.main(provider.java:8)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at com.intellij.rt.execution.application.AppMain.main(AppMain.java:140)
2015-05-25 21:05:24,120 [main] ERROR co.solinx.remote.zookeeper.ZookeeperClient (ZookeeperClient.java:87) - org.apache.curator.CuratorConnectionLossException: KeeperErrorCode = ConnectionLoss
2015-05-25 21:05:30,110 [main-SendThread(192.168.140.142:2181)] WARN  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:1102) - Session 0x0 for server null, unexpected error, closing socket connection and attempting reconnect
java.net.ConnectException: Connection timed out: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(Unknown Source)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:361)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1081)
2015-05-25 21:05:31,211 [main-SendThread(192.168.140.142:2181)] INFO  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:975) - Opening socket connection to server 192.168.140.142/192.168.140.142:2181. Will not attempt to authenticate using SASL (unknown error)
2015-05-25 21:05:39,137 [main] WARN  org.apache.curator.ConnectionState (ConnectionState.java:191) - Connection attempt unsuccessful after 68241 (greater than max timeout of 60000). Resetting connection and trying again with a new connection.
2015-05-25 21:05:52,328 [main] INFO  org.apache.zookeeper.ZooKeeper (ZooKeeper.java:684) - Session: 0x0 closed
2015-05-25 21:05:52,328 [main-EventThread] INFO  org.apache.zookeeper.ClientCnxn$EventThread (ClientCnxn.java:512) - EventThread shut down
2015-05-25 21:05:52,329 [main] INFO  org.apache.zookeeper.ZooKeeper (ZooKeeper.java:438) - Initiating client connection, connectString=192.168.140.142:2181 sessionTimeout=60000 watcher=org.apache.curator.ConnectionState@1a38c59b
2015-05-25 21:05:52,335 [main-SendThread(192.168.140.142:2181)] INFO  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:975) - Opening socket connection to server 192.168.140.142/192.168.140.142:2181. Will not attempt to authenticate using SASL (unknown error)
2015-05-25 21:06:13,338 [main-SendThread(192.168.140.142:2181)] WARN  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:1102) - Session 0x0 for server null, unexpected error, closing socket connection and attempting reconnect
java.net.ConnectException: Connection timed out: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(Unknown Source)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:361)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1081)
2015-05-25 21:06:14,440 [main-SendThread(192.168.140.142:2181)] INFO  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:975) - Opening socket connection to server 192.168.140.142/192.168.140.142:2181. Will not attempt to authenticate using SASL (unknown error)
2015-05-25 21:06:29,453 [main] ERROR org.apache.curator.ConnectionState (ConnectionState.java:200) - Connection timed out for connection string (192.168.140.142:2181) and timeout (15000) / elapsed (50315)
org.apache.curator.CuratorConnectionLossException: KeeperErrorCode = ConnectionLoss
	at org.apache.curator.ConnectionState.checkTimeouts(ConnectionState.java:197)
	at org.apache.curator.ConnectionState.getZooKeeper(ConnectionState.java:87)
	at org.apache.curator.CuratorZookeeperClient.getZooKeeper(CuratorZookeeperClient.java:115)
	at org.apache.curator.framework.imps.CuratorFrameworkImpl.getZooKeeper(CuratorFrameworkImpl.java:477)
	at org.apache.curator.framework.imps.ExistsBuilderImpl$2.call(ExistsBuilderImpl.java:172)
	at org.apache.curator.framework.imps.ExistsBuilderImpl$2.call(ExistsBuilderImpl.java:161)
	at org.apache.curator.RetryLoop.callWithRetry(RetryLoop.java:107)
	at org.apache.curator.framework.imps.ExistsBuilderImpl.pathInForeground(ExistsBuilderImpl.java:158)
	at org.apache.curator.framework.imps.ExistsBuilderImpl.forPath(ExistsBuilderImpl.java:148)
	at org.apache.curator.framework.imps.ExistsBuilderImpl.forPath(ExistsBuilderImpl.java:36)
	at co.solinx.remote.zookeeper.ZookeeperClient.checkExists(ZookeeperClient.java:135)
	at co.solinx.remote.zookeeper.ZookeeperClient.createNote(ZookeeperClient.java:72)
	at co.solinx.forest.registry.zookeeper.ZookeeperRegistry.registryServerApi(ZookeeperRegistry.java:78)
	at co.solinx.forest.registry.zookeeper.ZookeeperRegistry.registryService(ZookeeperRegistry.java:137)
	at co.solinx.forest.config.ServiceConfig.Export(ServiceConfig.java:54)
	at co.solinx.forest.config.spring.ServiceBean.onApplicationEvent(ServiceBean.java:52)
	at org.springframework.context.event.SimpleApplicationEventMulticaster.invokeListener(SimpleApplicationEventMulticaster.java:151)
	at org.springframework.context.event.SimpleApplicationEventMulticaster.multicastEvent(SimpleApplicationEventMulticaster.java:128)
	at org.springframework.context.support.AbstractApplicationContext.publishEvent(AbstractApplicationContext.java:331)
	at org.springframework.context.support.AbstractApplicationContext.finishRefresh(AbstractApplicationContext.java:773)
	at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:483)
	at org.springframework.context.support.ClassPathXmlApplicationContext.<init>(ClassPathXmlApplicationContext.java:139)
	at org.springframework.context.support.ClassPathXmlApplicationContext.<init>(ClassPathXmlApplicationContext.java:83)
	at co.solinx.forest.container.spring.SpringContainer.start(SpringContainer.java:23)
	at co.solinx.forest.container.Main.main(Main.java:29)
	at provider.main(provider.java:8)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at com.intellij.rt.execution.application.AppMain.main(AppMain.java:140)
2015-05-25 21:06:35,435 [main-SendThread(192.168.140.142:2181)] WARN  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:1102) - Session 0x0 for server null, unexpected error, closing socket connection and attempting reconnect
java.net.ConnectException: Connection timed out: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(Unknown Source)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:361)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1081)
2015-05-25 21:06:36,535 [main-SendThread(192.168.140.142:2181)] INFO  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:975) - Opening socket connection to server 192.168.140.142/192.168.140.142:2181. Will not attempt to authenticate using SASL (unknown error)
2015-05-25 21:06:47,466 [main] WARN  org.apache.curator.ConnectionState (ConnectionState.java:191) - Connection attempt unsuccessful after 68328 (greater than max timeout of 60000). Resetting connection and trying again with a new connection.
2015-05-25 21:06:57,636 [main] INFO  org.apache.zookeeper.ZooKeeper (ZooKeeper.java:684) - Session: 0x0 closed
2015-05-25 21:06:57,636 [main-EventThread] INFO  org.apache.zookeeper.ClientCnxn$EventThread (ClientCnxn.java:512) - EventThread shut down
2015-05-25 21:06:57,637 [main] INFO  org.apache.zookeeper.ZooKeeper (ZooKeeper.java:438) - Initiating client connection, connectString=192.168.140.142:2181 sessionTimeout=60000 watcher=org.apache.curator.ConnectionState@1a38c59b
2015-05-25 21:06:57,643 [main-SendThread(192.168.140.142:2181)] INFO  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:975) - Opening socket connection to server 192.168.140.142/192.168.140.142:2181. Will not attempt to authenticate using SASL (unknown error)
2015-05-25 21:07:18,645 [main-SendThread(192.168.140.142:2181)] WARN  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:1102) - Session 0x0 for server null, unexpected error, closing socket connection and attempting reconnect
java.net.ConnectException: Connection timed out: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(Unknown Source)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:361)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1081)
2015-05-25 21:07:19,747 [main-SendThread(192.168.140.142:2181)] INFO  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:975) - Opening socket connection to server 192.168.140.142/192.168.140.142:2181. Will not attempt to authenticate using SASL (unknown error)
2015-05-25 21:07:37,759 [main] ERROR org.apache.curator.ConnectionState (ConnectionState.java:200) - Connection timed out for connection string (192.168.140.142:2181) and timeout (15000) / elapsed (50292)
org.apache.curator.CuratorConnectionLossException: KeeperErrorCode = ConnectionLoss
	at org.apache.curator.ConnectionState.checkTimeouts(ConnectionState.java:197)
	at org.apache.curator.ConnectionState.getZooKeeper(ConnectionState.java:87)
	at org.apache.curator.CuratorZookeeperClient.getZooKeeper(CuratorZookeeperClient.java:115)
	at org.apache.curator.framework.imps.CuratorFrameworkImpl.getZooKeeper(CuratorFrameworkImpl.java:477)
	at org.apache.curator.framework.imps.ExistsBuilderImpl$2.call(ExistsBuilderImpl.java:172)
	at org.apache.curator.framework.imps.ExistsBuilderImpl$2.call(ExistsBuilderImpl.java:161)
	at org.apache.curator.RetryLoop.callWithRetry(RetryLoop.java:107)
	at org.apache.curator.framework.imps.ExistsBuilderImpl.pathInForeground(ExistsBuilderImpl.java:158)
	at org.apache.curator.framework.imps.ExistsBuilderImpl.forPath(ExistsBuilderImpl.java:148)
	at org.apache.curator.framework.imps.ExistsBuilderImpl.forPath(ExistsBuilderImpl.java:36)
	at co.solinx.remote.zookeeper.ZookeeperClient.checkExists(ZookeeperClient.java:135)
	at co.solinx.remote.zookeeper.ZookeeperClient.createNote(ZookeeperClient.java:72)
	at co.solinx.forest.registry.zookeeper.ZookeeperRegistry.registryServerApi(ZookeeperRegistry.java:78)
	at co.solinx.forest.registry.zookeeper.ZookeeperRegistry.registryService(ZookeeperRegistry.java:137)
	at co.solinx.forest.config.ServiceConfig.Export(ServiceConfig.java:54)
	at co.solinx.forest.config.spring.ServiceBean.onApplicationEvent(ServiceBean.java:52)
	at org.springframework.context.event.SimpleApplicationEventMulticaster.invokeListener(SimpleApplicationEventMulticaster.java:151)
	at org.springframework.context.event.SimpleApplicationEventMulticaster.multicastEvent(SimpleApplicationEventMulticaster.java:128)
	at org.springframework.context.support.AbstractApplicationContext.publishEvent(AbstractApplicationContext.java:331)
	at org.springframework.context.support.AbstractApplicationContext.finishRefresh(AbstractApplicationContext.java:773)
	at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:483)
	at org.springframework.context.support.ClassPathXmlApplicationContext.<init>(ClassPathXmlApplicationContext.java:139)
	at org.springframework.context.support.ClassPathXmlApplicationContext.<init>(ClassPathXmlApplicationContext.java:83)
	at co.solinx.forest.container.spring.SpringContainer.start(SpringContainer.java:23)
	at co.solinx.forest.container.Main.main(Main.java:29)
	at provider.main(provider.java:8)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at com.intellij.rt.execution.application.AppMain.main(AppMain.java:140)
2015-05-25 21:07:37,760 [main] ERROR co.solinx.remote.zookeeper.ZookeeperClient (ZookeeperClient.java:87) - org.apache.curator.CuratorConnectionLossException: KeeperErrorCode = ConnectionLoss
2015-05-25 21:07:40,750 [main-SendThread(192.168.140.142:2181)] WARN  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:1102) - Session 0x0 for server null, unexpected error, closing socket connection and attempting reconnect
java.net.ConnectException: Connection timed out: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(Unknown Source)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:361)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1081)
2015-05-25 21:07:41,852 [main-SendThread(192.168.140.142:2181)] INFO  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:975) - Opening socket connection to server 192.168.140.142/192.168.140.142:2181. Will not attempt to authenticate using SASL (unknown error)
2015-05-25 21:07:52,774 [main] WARN  org.apache.curator.ConnectionState (ConnectionState.java:191) - Connection attempt unsuccessful after 65307 (greater than max timeout of 60000). Resetting connection and trying again with a new connection.
2015-05-25 21:08:02,953 [main] INFO  org.apache.zookeeper.ZooKeeper (ZooKeeper.java:684) - Session: 0x0 closed
2015-05-25 21:08:02,953 [main-EventThread] INFO  org.apache.zookeeper.ClientCnxn$EventThread (ClientCnxn.java:512) - EventThread shut down
2015-05-25 21:08:02,954 [main] INFO  org.apache.zookeeper.ZooKeeper (ZooKeeper.java:438) - Initiating client connection, connectString=192.168.140.142:2181 sessionTimeout=60000 watcher=org.apache.curator.ConnectionState@1a38c59b
2015-05-25 21:08:02,962 [main-SendThread(192.168.140.142:2181)] INFO  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:975) - Opening socket connection to server 192.168.140.142/192.168.140.142:2181. Will not attempt to authenticate using SASL (unknown error)
2015-05-25 21:08:23,964 [main-SendThread(192.168.140.142:2181)] WARN  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:1102) - Session 0x0 for server null, unexpected error, closing socket connection and attempting reconnect
java.net.ConnectException: Connection timed out: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(Unknown Source)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:361)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1081)
2015-05-25 21:08:25,066 [main-SendThread(192.168.140.142:2181)] INFO  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:975) - Opening socket connection to server 192.168.140.142/192.168.140.142:2181. Will not attempt to authenticate using SASL (unknown error)
2015-05-25 21:08:40,080 [main] ERROR org.apache.curator.ConnectionState (ConnectionState.java:200) - Connection timed out for connection string (192.168.140.142:2181) and timeout (15000) / elapsed (47305)
org.apache.curator.CuratorConnectionLossException: KeeperErrorCode = ConnectionLoss
	at org.apache.curator.ConnectionState.checkTimeouts(ConnectionState.java:197)
	at org.apache.curator.ConnectionState.getZooKeeper(ConnectionState.java:87)
	at org.apache.curator.CuratorZookeeperClient.getZooKeeper(CuratorZookeeperClient.java:115)
	at org.apache.curator.framework.imps.CuratorFrameworkImpl.getZooKeeper(CuratorFrameworkImpl.java:477)
	at org.apache.curator.framework.imps.ExistsBuilderImpl$2.call(ExistsBuilderImpl.java:172)
	at org.apache.curator.framework.imps.ExistsBuilderImpl$2.call(ExistsBuilderImpl.java:161)
	at org.apache.curator.RetryLoop.callWithRetry(RetryLoop.java:107)
	at org.apache.curator.framework.imps.ExistsBuilderImpl.pathInForeground(ExistsBuilderImpl.java:158)
	at org.apache.curator.framework.imps.ExistsBuilderImpl.forPath(ExistsBuilderImpl.java:148)
	at org.apache.curator.framework.imps.ExistsBuilderImpl.forPath(ExistsBuilderImpl.java:36)
	at co.solinx.remote.zookeeper.ZookeeperClient.checkExists(ZookeeperClient.java:135)
	at co.solinx.remote.zookeeper.ZookeeperClient.createNote(ZookeeperClient.java:72)
	at co.solinx.forest.registry.zookeeper.ZookeeperRegistry.createProviderNote(ZookeeperRegistry.java:106)
	at co.solinx.forest.registry.zookeeper.ZookeeperRegistry.registryService(ZookeeperRegistry.java:138)
	at co.solinx.forest.config.ServiceConfig.Export(ServiceConfig.java:54)
	at co.solinx.forest.config.spring.ServiceBean.onApplicationEvent(ServiceBean.java:52)
	at org.springframework.context.event.SimpleApplicationEventMulticaster.invokeListener(SimpleApplicationEventMulticaster.java:151)
	at org.springframework.context.event.SimpleApplicationEventMulticaster.multicastEvent(SimpleApplicationEventMulticaster.java:128)
	at org.springframework.context.support.AbstractApplicationContext.publishEvent(AbstractApplicationContext.java:331)
	at org.springframework.context.support.AbstractApplicationContext.finishRefresh(AbstractApplicationContext.java:773)
	at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:483)
	at org.springframework.context.support.ClassPathXmlApplicationContext.<init>(ClassPathXmlApplicationContext.java:139)
	at org.springframework.context.support.ClassPathXmlApplicationContext.<init>(ClassPathXmlApplicationContext.java:83)
	at co.solinx.forest.container.spring.SpringContainer.start(SpringContainer.java:23)
	at co.solinx.forest.container.Main.main(Main.java:29)
	at provider.main(provider.java:8)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at com.intellij.rt.execution.application.AppMain.main(AppMain.java:140)
2015-05-25 21:08:46,078 [main-SendThread(192.168.140.142:2181)] WARN  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:1102) - Session 0x0 for server null, unexpected error, closing socket connection and attempting reconnect
java.net.ConnectException: Connection timed out: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(Unknown Source)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:361)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1081)
2015-05-25 21:08:47,180 [main-SendThread(192.168.140.142:2181)] INFO  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:975) - Opening socket connection to server 192.168.140.142/192.168.140.142:2181. Will not attempt to authenticate using SASL (unknown error)
2015-05-25 21:08:56,095 [main] WARN  org.apache.curator.ConnectionState (ConnectionState.java:191) - Connection attempt unsuccessful after 63320 (greater than max timeout of 60000). Resetting connection and trying again with a new connection.
2015-05-25 21:09:08,284 [main] INFO  org.apache.zookeeper.ZooKeeper (ZooKeeper.java:684) - Session: 0x0 closed
2015-05-25 21:09:08,284 [main-EventThread] INFO  org.apache.zookeeper.ClientCnxn$EventThread (ClientCnxn.java:512) - EventThread shut down
2015-05-25 21:09:08,285 [main] INFO  org.apache.zookeeper.ZooKeeper (ZooKeeper.java:438) - Initiating client connection, connectString=192.168.140.142:2181 sessionTimeout=60000 watcher=org.apache.curator.ConnectionState@1a38c59b
2015-05-25 21:09:08,291 [main-SendThread(192.168.140.142:2181)] INFO  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:975) - Opening socket connection to server 192.168.140.142/192.168.140.142:2181. Will not attempt to authenticate using SASL (unknown error)
2015-05-25 21:09:29,296 [main-SendThread(192.168.140.142:2181)] WARN  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:1102) - Session 0x0 for server null, unexpected error, closing socket connection and attempting reconnect
java.net.ConnectException: Connection timed out: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(Unknown Source)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:361)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1081)
2015-05-25 21:09:30,398 [main-SendThread(192.168.140.142:2181)] INFO  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:975) - Opening socket connection to server 192.168.140.142/192.168.140.142:2181. Will not attempt to authenticate using SASL (unknown error)
2015-05-25 21:09:50,412 [main] ERROR org.apache.curator.ConnectionState (ConnectionState.java:200) - Connection timed out for connection string (192.168.140.142:2181) and timeout (15000) / elapsed (54316)
org.apache.curator.CuratorConnectionLossException: KeeperErrorCode = ConnectionLoss
	at org.apache.curator.ConnectionState.checkTimeouts(ConnectionState.java:197)
	at org.apache.curator.ConnectionState.getZooKeeper(ConnectionState.java:87)
	at org.apache.curator.CuratorZookeeperClient.getZooKeeper(CuratorZookeeperClient.java:115)
	at org.apache.curator.framework.imps.CuratorFrameworkImpl.getZooKeeper(CuratorFrameworkImpl.java:477)
	at org.apache.curator.framework.imps.ExistsBuilderImpl$2.call(ExistsBuilderImpl.java:172)
	at org.apache.curator.framework.imps.ExistsBuilderImpl$2.call(ExistsBuilderImpl.java:161)
	at org.apache.curator.RetryLoop.callWithRetry(RetryLoop.java:107)
	at org.apache.curator.framework.imps.ExistsBuilderImpl.pathInForeground(ExistsBuilderImpl.java:158)
	at org.apache.curator.framework.imps.ExistsBuilderImpl.forPath(ExistsBuilderImpl.java:148)
	at org.apache.curator.framework.imps.ExistsBuilderImpl.forPath(ExistsBuilderImpl.java:36)
	at co.solinx.remote.zookeeper.ZookeeperClient.checkExists(ZookeeperClient.java:135)
	at co.solinx.remote.zookeeper.ZookeeperClient.createNote(ZookeeperClient.java:72)
	at co.solinx.forest.registry.zookeeper.ZookeeperRegistry.createProviderNote(ZookeeperRegistry.java:106)
	at co.solinx.forest.registry.zookeeper.ZookeeperRegistry.registryService(ZookeeperRegistry.java:138)
	at co.solinx.forest.config.ServiceConfig.Export(ServiceConfig.java:54)
	at co.solinx.forest.config.spring.ServiceBean.onApplicationEvent(ServiceBean.java:52)
	at org.springframework.context.event.SimpleApplicationEventMulticaster.invokeListener(SimpleApplicationEventMulticaster.java:151)
	at org.springframework.context.event.SimpleApplicationEventMulticaster.multicastEvent(SimpleApplicationEventMulticaster.java:128)
	at org.springframework.context.support.AbstractApplicationContext.publishEvent(AbstractApplicationContext.java:331)
	at org.springframework.context.support.AbstractApplicationContext.finishRefresh(AbstractApplicationContext.java:773)
	at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:483)
	at org.springframework.context.support.ClassPathXmlApplicationContext.<init>(ClassPathXmlApplicationContext.java:139)
	at org.springframework.context.support.ClassPathXmlApplicationContext.<init>(ClassPathXmlApplicationContext.java:83)
	at co.solinx.forest.container.spring.SpringContainer.start(SpringContainer.java:23)
	at co.solinx.forest.container.Main.main(Main.java:29)
	at provider.main(provider.java:8)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at com.intellij.rt.execution.application.AppMain.main(AppMain.java:140)
2015-05-25 21:09:50,414 [main] ERROR co.solinx.remote.zookeeper.ZookeeperClient (ZookeeperClient.java:87) - org.apache.curator.CuratorConnectionLossException: KeeperErrorCode = ConnectionLoss
2015-05-25 21:09:51,410 [main-SendThread(192.168.140.142:2181)] WARN  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:1102) - Session 0x0 for server null, unexpected error, closing socket connection and attempting reconnect
java.net.ConnectException: Connection timed out: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(Unknown Source)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:361)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1081)
2015-05-25 21:09:52,511 [main-SendThread(192.168.140.142:2181)] INFO  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:975) - Opening socket connection to server 192.168.140.142/192.168.140.142:2181. Will not attempt to authenticate using SASL (unknown error)
2015-05-25 21:10:05,468 [main] WARN  org.apache.curator.ConnectionState (ConnectionState.java:191) - Connection attempt unsuccessful after 69372 (greater than max timeout of 60000). Resetting connection and trying again with a new connection.
2015-05-25 21:10:13,621 [main] INFO  org.apache.zookeeper.ZooKeeper (ZooKeeper.java:684) - Session: 0x0 closed
2015-05-25 21:10:13,621 [main-EventThread] INFO  org.apache.zookeeper.ClientCnxn$EventThread (ClientCnxn.java:512) - EventThread shut down
2015-05-25 21:10:13,622 [main] INFO  org.apache.zookeeper.ZooKeeper (ZooKeeper.java:438) - Initiating client connection, connectString=192.168.140.142:2181 sessionTimeout=60000 watcher=org.apache.curator.ConnectionState@1a38c59b
2015-05-25 21:10:13,629 [main-SendThread(192.168.140.142:2181)] INFO  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:975) - Opening socket connection to server 192.168.140.142/192.168.140.142:2181. Will not attempt to authenticate using SASL (unknown error)
2015-05-25 21:10:34,635 [main-SendThread(192.168.140.142:2181)] WARN  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:1102) - Session 0x0 for server null, unexpected error, closing socket connection and attempting reconnect
java.net.ConnectException: Connection timed out: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(Unknown Source)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:361)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1081)
2015-05-25 21:10:35,737 [main-SendThread(192.168.140.142:2181)] INFO  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:975) - Opening socket connection to server 192.168.140.142/192.168.140.142:2181. Will not attempt to authenticate using SASL (unknown error)
2015-05-25 21:10:50,755 [main] ERROR org.apache.curator.ConnectionState (ConnectionState.java:200) - Connection timed out for connection string (192.168.140.142:2181) and timeout (15000) / elapsed (45286)
org.apache.curator.CuratorConnectionLossException: KeeperErrorCode = ConnectionLoss
	at org.apache.curator.ConnectionState.checkTimeouts(ConnectionState.java:197)
	at org.apache.curator.ConnectionState.getZooKeeper(ConnectionState.java:87)
	at org.apache.curator.CuratorZookeeperClient.getZooKeeper(CuratorZookeeperClient.java:115)
	at org.apache.curator.framework.imps.CuratorFrameworkImpl.getZooKeeper(CuratorFrameworkImpl.java:477)
	at org.apache.curator.framework.imps.ExistsBuilderImpl$2.call(ExistsBuilderImpl.java:172)
	at org.apache.curator.framework.imps.ExistsBuilderImpl$2.call(ExistsBuilderImpl.java:161)
	at org.apache.curator.RetryLoop.callWithRetry(RetryLoop.java:107)
	at org.apache.curator.framework.imps.ExistsBuilderImpl.pathInForeground(ExistsBuilderImpl.java:158)
	at org.apache.curator.framework.imps.ExistsBuilderImpl.forPath(ExistsBuilderImpl.java:148)
	at org.apache.curator.framework.imps.ExistsBuilderImpl.forPath(ExistsBuilderImpl.java:36)
	at co.solinx.remote.zookeeper.ZookeeperClient.checkExists(ZookeeperClient.java:135)
	at co.solinx.remote.zookeeper.ZookeeperClient.createNote(ZookeeperClient.java:72)
	at co.solinx.forest.registry.zookeeper.ZookeeperRegistry.registryService(ZookeeperRegistry.java:146)
	at co.solinx.forest.config.ServiceConfig.Export(ServiceConfig.java:54)
	at co.solinx.forest.config.spring.ServiceBean.onApplicationEvent(ServiceBean.java:52)
	at org.springframework.context.event.SimpleApplicationEventMulticaster.invokeListener(SimpleApplicationEventMulticaster.java:151)
	at org.springframework.context.event.SimpleApplicationEventMulticaster.multicastEvent(SimpleApplicationEventMulticaster.java:128)
	at org.springframework.context.support.AbstractApplicationContext.publishEvent(AbstractApplicationContext.java:331)
	at org.springframework.context.support.AbstractApplicationContext.finishRefresh(AbstractApplicationContext.java:773)
	at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:483)
	at org.springframework.context.support.ClassPathXmlApplicationContext.<init>(ClassPathXmlApplicationContext.java:139)
	at org.springframework.context.support.ClassPathXmlApplicationContext.<init>(ClassPathXmlApplicationContext.java:83)
	at co.solinx.forest.container.spring.SpringContainer.start(SpringContainer.java:23)
	at co.solinx.forest.container.Main.main(Main.java:29)
	at provider.main(provider.java:8)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at com.intellij.rt.execution.application.AppMain.main(AppMain.java:140)
2015-05-25 21:10:56,736 [main-SendThread(192.168.140.142:2181)] WARN  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:1102) - Session 0x0 for server null, unexpected error, closing socket connection and attempting reconnect
java.net.ConnectException: Connection timed out: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(Unknown Source)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:361)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1081)
2015-05-25 21:10:57,838 [main-SendThread(192.168.140.142:2181)] INFO  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:975) - Opening socket connection to server 192.168.140.142/192.168.140.142:2181. Will not attempt to authenticate using SASL (unknown error)
2015-05-25 21:11:06,770 [main] WARN  org.apache.curator.ConnectionState (ConnectionState.java:191) - Connection attempt unsuccessful after 61301 (greater than max timeout of 60000). Resetting connection and trying again with a new connection.
2015-05-25 21:11:18,946 [main] INFO  org.apache.zookeeper.ZooKeeper (ZooKeeper.java:684) - Session: 0x0 closed
2015-05-25 21:11:18,946 [main-EventThread] INFO  org.apache.zookeeper.ClientCnxn$EventThread (ClientCnxn.java:512) - EventThread shut down
2015-05-25 21:11:18,947 [main] INFO  org.apache.zookeeper.ZooKeeper (ZooKeeper.java:438) - Initiating client connection, connectString=192.168.140.142:2181 sessionTimeout=60000 watcher=org.apache.curator.ConnectionState@1a38c59b
2015-05-25 21:11:18,952 [main-SendThread(192.168.140.142:2181)] INFO  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:975) - Opening socket connection to server 192.168.140.142/192.168.140.142:2181. Will not attempt to authenticate using SASL (unknown error)
2015-05-25 21:11:39,960 [main-SendThread(192.168.140.142:2181)] WARN  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:1102) - Session 0x0 for server null, unexpected error, closing socket connection and attempting reconnect
java.net.ConnectException: Connection timed out: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(Unknown Source)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:361)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1081)
2015-05-25 21:11:41,061 [main-SendThread(192.168.140.142:2181)] INFO  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:975) - Opening socket connection to server 192.168.140.142/192.168.140.142:2181. Will not attempt to authenticate using SASL (unknown error)
2015-05-25 21:11:59,075 [main] ERROR org.apache.curator.ConnectionState (ConnectionState.java:200) - Connection timed out for connection string (192.168.140.142:2181) and timeout (15000) / elapsed (52305)
org.apache.curator.CuratorConnectionLossException: KeeperErrorCode = ConnectionLoss
	at org.apache.curator.ConnectionState.checkTimeouts(ConnectionState.java:197)
	at org.apache.curator.ConnectionState.getZooKeeper(ConnectionState.java:87)
	at org.apache.curator.CuratorZookeeperClient.getZooKeeper(CuratorZookeeperClient.java:115)
	at org.apache.curator.framework.imps.CuratorFrameworkImpl.getZooKeeper(CuratorFrameworkImpl.java:477)
	at org.apache.curator.framework.imps.ExistsBuilderImpl$2.call(ExistsBuilderImpl.java:172)
	at org.apache.curator.framework.imps.ExistsBuilderImpl$2.call(ExistsBuilderImpl.java:161)
	at org.apache.curator.RetryLoop.callWithRetry(RetryLoop.java:107)
	at org.apache.curator.framework.imps.ExistsBuilderImpl.pathInForeground(ExistsBuilderImpl.java:158)
	at org.apache.curator.framework.imps.ExistsBuilderImpl.forPath(ExistsBuilderImpl.java:148)
	at org.apache.curator.framework.imps.ExistsBuilderImpl.forPath(ExistsBuilderImpl.java:36)
	at co.solinx.remote.zookeeper.ZookeeperClient.checkExists(ZookeeperClient.java:135)
	at co.solinx.remote.zookeeper.ZookeeperClient.createNote(ZookeeperClient.java:72)
	at co.solinx.forest.registry.zookeeper.ZookeeperRegistry.registryService(ZookeeperRegistry.java:146)
	at co.solinx.forest.config.ServiceConfig.Export(ServiceConfig.java:54)
	at co.solinx.forest.config.spring.ServiceBean.onApplicationEvent(ServiceBean.java:52)
	at org.springframework.context.event.SimpleApplicationEventMulticaster.invokeListener(SimpleApplicationEventMulticaster.java:151)
	at org.springframework.context.event.SimpleApplicationEventMulticaster.multicastEvent(SimpleApplicationEventMulticaster.java:128)
	at org.springframework.context.support.AbstractApplicationContext.publishEvent(AbstractApplicationContext.java:331)
	at org.springframework.context.support.AbstractApplicationContext.finishRefresh(AbstractApplicationContext.java:773)
	at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:483)
	at org.springframework.context.support.ClassPathXmlApplicationContext.<init>(ClassPathXmlApplicationContext.java:139)
	at org.springframework.context.support.ClassPathXmlApplicationContext.<init>(ClassPathXmlApplicationContext.java:83)
	at co.solinx.forest.container.spring.SpringContainer.start(SpringContainer.java:23)
	at co.solinx.forest.container.Main.main(Main.java:29)
	at provider.main(provider.java:8)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at com.intellij.rt.execution.application.AppMain.main(AppMain.java:140)
2015-05-25 21:11:59,079 [main] ERROR co.solinx.remote.zookeeper.ZookeeperClient (ZookeeperClient.java:87) - org.apache.curator.CuratorConnectionLossException: KeeperErrorCode = ConnectionLoss
2015-05-25 21:11:59,079 [main] INFO  co.solinx.forest.config.ServiceConfig (ServiceConfig.java:55) - [co.solinx.forest.demo.impl.HelloForestServiceImpl@4b5d6a01]
2015-05-25 21:11:59,080 [main] INFO  co.solinx.forest.config.spring.ServiceBean (ServiceBean.java:51) - _________________________org.springframework.context.event.ContextRefreshedEvent[source=org.springframework.context.support.ClassPathXmlApplicationContext@7fbe847c: startup date [Mon May 25 21:03:19 CST 2015]; root of context hierarchy]
2015-05-25 21:11:59,080 [main] INFO  co.solinx.forest.config.ServiceConfig (ServiceConfig.java:40) - Export£ºco.solinx.forest.demo.api.IHelloRpcService
2015-05-25 21:11:59,081 [main] INFO  co.solinx.forest.config.ServiceConfig (ServiceConfig.java:45) - interfaceName_______________co.solinx.forest.demo.api.IHelloRpcService
2015-05-25 21:12:02,065 [main-SendThread(192.168.140.142:2181)] WARN  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:1102) - Session 0x0 for server null, unexpected error, closing socket connection and attempting reconnect
java.net.ConnectException: Connection timed out: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(Unknown Source)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:361)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1081)
2015-05-25 21:12:03,165 [main-SendThread(192.168.140.142:2181)] INFO  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:975) - Opening socket connection to server 192.168.140.142/192.168.140.142:2181. Will not attempt to authenticate using SASL (unknown error)
2015-05-25 21:12:14,095 [main] WARN  org.apache.curator.ConnectionState (ConnectionState.java:191) - Connection attempt unsuccessful after 67325 (greater than max timeout of 60000). Resetting connection and trying again with a new connection.
2015-05-25 21:12:24,264 [main] INFO  org.apache.zookeeper.ZooKeeper (ZooKeeper.java:684) - Session: 0x0 closed
2015-05-25 21:12:24,264 [main-EventThread] INFO  org.apache.zookeeper.ClientCnxn$EventThread (ClientCnxn.java:512) - EventThread shut down
2015-05-25 21:12:24,265 [main] INFO  org.apache.zookeeper.ZooKeeper (ZooKeeper.java:438) - Initiating client connection, connectString=192.168.140.142:2181 sessionTimeout=60000 watcher=org.apache.curator.ConnectionState@1a38c59b
2015-05-25 21:12:24,269 [main-SendThread(192.168.140.142:2181)] INFO  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:975) - Opening socket connection to server 192.168.140.142/192.168.140.142:2181. Will not attempt to authenticate using SASL (unknown error)
2015-05-25 21:12:45,272 [main-SendThread(192.168.140.142:2181)] WARN  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:1102) - Session 0x0 for server null, unexpected error, closing socket connection and attempting reconnect
java.net.ConnectException: Connection timed out: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(Unknown Source)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:361)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1081)
2015-05-25 21:12:46,374 [main-SendThread(192.168.140.142:2181)] INFO  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:975) - Opening socket connection to server 192.168.140.142/192.168.140.142:2181. Will not attempt to authenticate using SASL (unknown error)
2015-05-25 21:13:01,387 [main] ERROR org.apache.curator.ConnectionState (ConnectionState.java:200) - Connection timed out for connection string (192.168.140.142:2181) and timeout (15000) / elapsed (47292)
org.apache.curator.CuratorConnectionLossException: KeeperErrorCode = ConnectionLoss
	at org.apache.curator.ConnectionState.checkTimeouts(ConnectionState.java:197)
	at org.apache.curator.ConnectionState.getZooKeeper(ConnectionState.java:87)
	at org.apache.curator.CuratorZookeeperClient.getZooKeeper(CuratorZookeeperClient.java:115)
	at org.apache.curator.framework.imps.CuratorFrameworkImpl.getZooKeeper(CuratorFrameworkImpl.java:477)
	at org.apache.curator.framework.imps.ExistsBuilderImpl$2.call(ExistsBuilderImpl.java:172)
	at org.apache.curator.framework.imps.ExistsBuilderImpl$2.call(ExistsBuilderImpl.java:161)
	at org.apache.curator.RetryLoop.callWithRetry(RetryLoop.java:107)
	at org.apache.curator.framework.imps.ExistsBuilderImpl.pathInForeground(ExistsBuilderImpl.java:158)
	at org.apache.curator.framework.imps.ExistsBuilderImpl.forPath(ExistsBuilderImpl.java:148)
	at org.apache.curator.framework.imps.ExistsBuilderImpl.forPath(ExistsBuilderImpl.java:36)
	at co.solinx.remote.zookeeper.ZookeeperClient.checkExists(ZookeeperClient.java:135)
	at co.solinx.remote.zookeeper.ZookeeperClient.createNote(ZookeeperClient.java:72)
	at co.solinx.forest.registry.zookeeper.ZookeeperRegistry.createRoot(ZookeeperRegistry.java:68)
	at co.solinx.forest.registry.zookeeper.ZookeeperRegistry.registryService(ZookeeperRegistry.java:136)
	at co.solinx.forest.config.ServiceConfig.Export(ServiceConfig.java:54)
	at co.solinx.forest.config.spring.ServiceBean.onApplicationEvent(ServiceBean.java:52)
	at org.springframework.context.event.SimpleApplicationEventMulticaster.invokeListener(SimpleApplicationEventMulticaster.java:151)
	at org.springframework.context.event.SimpleApplicationEventMulticaster.multicastEvent(SimpleApplicationEventMulticaster.java:128)
	at org.springframework.context.support.AbstractApplicationContext.publishEvent(AbstractApplicationContext.java:331)
	at org.springframework.context.support.AbstractApplicationContext.finishRefresh(AbstractApplicationContext.java:773)
	at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:483)
	at org.springframework.context.support.ClassPathXmlApplicationContext.<init>(ClassPathXmlApplicationContext.java:139)
	at org.springframework.context.support.ClassPathXmlApplicationContext.<init>(ClassPathXmlApplicationContext.java:83)
	at co.solinx.forest.container.spring.SpringContainer.start(SpringContainer.java:23)
	at co.solinx.forest.container.Main.main(Main.java:29)
	at provider.main(provider.java:8)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at com.intellij.rt.execution.application.AppMain.main(AppMain.java:140)
2015-05-25 21:13:07,375 [main-SendThread(192.168.140.142:2181)] WARN  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:1102) - Session 0x0 for server null, unexpected error, closing socket connection and attempting reconnect
java.net.ConnectException: Connection timed out: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(Unknown Source)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:361)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1081)
2015-05-25 21:13:08,476 [main-SendThread(192.168.140.142:2181)] INFO  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:975) - Opening socket connection to server 192.168.140.142/192.168.140.142:2181. Will not attempt to authenticate using SASL (unknown error)
2015-05-25 21:13:18,401 [main] WARN  org.apache.curator.ConnectionState (ConnectionState.java:191) - Connection attempt unsuccessful after 64306 (greater than max timeout of 60000). Resetting connection and trying again with a new connection.
2015-05-25 21:13:29,578 [main] INFO  org.apache.zookeeper.ZooKeeper (ZooKeeper.java:684) - Session: 0x0 closed
2015-05-25 21:13:29,578 [main-EventThread] INFO  org.apache.zookeeper.ClientCnxn$EventThread (ClientCnxn.java:512) - EventThread shut down
2015-05-25 21:13:29,579 [main] INFO  org.apache.zookeeper.ZooKeeper (ZooKeeper.java:438) - Initiating client connection, connectString=192.168.140.142:2181 sessionTimeout=60000 watcher=org.apache.curator.ConnectionState@1a38c59b
2015-05-25 21:13:29,582 [main-SendThread(192.168.140.142:2181)] INFO  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:975) - Opening socket connection to server 192.168.140.142/192.168.140.142:2181. Will not attempt to authenticate using SASL (unknown error)
2015-05-25 21:13:50,594 [main-SendThread(192.168.140.142:2181)] WARN  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:1102) - Session 0x0 for server null, unexpected error, closing socket connection and attempting reconnect
java.net.ConnectException: Connection timed out: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(Unknown Source)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:361)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1081)
2015-05-25 21:13:51,696 [main-SendThread(192.168.140.142:2181)] INFO  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:975) - Opening socket connection to server 192.168.140.142/192.168.140.142:2181. Will not attempt to authenticate using SASL (unknown error)
2015-05-25 21:14:06,711 [main] ERROR org.apache.curator.ConnectionState (ConnectionState.java:200) - Connection timed out for connection string (192.168.140.142:2181) and timeout (15000) / elapsed (48310)
org.apache.curator.CuratorConnectionLossException: KeeperErrorCode = ConnectionLoss
	at org.apache.curator.ConnectionState.checkTimeouts(ConnectionState.java:197)
	at org.apache.curator.ConnectionState.getZooKeeper(ConnectionState.java:87)
	at org.apache.curator.CuratorZookeeperClient.getZooKeeper(CuratorZookeeperClient.java:115)
	at org.apache.curator.framework.imps.CuratorFrameworkImpl.getZooKeeper(CuratorFrameworkImpl.java:477)
	at org.apache.curator.framework.imps.ExistsBuilderImpl$2.call(ExistsBuilderImpl.java:172)
	at org.apache.curator.framework.imps.ExistsBuilderImpl$2.call(ExistsBuilderImpl.java:161)
	at org.apache.curator.RetryLoop.callWithRetry(RetryLoop.java:107)
	at org.apache.curator.framework.imps.ExistsBuilderImpl.pathInForeground(ExistsBuilderImpl.java:158)
	at org.apache.curator.framework.imps.ExistsBuilderImpl.forPath(ExistsBuilderImpl.java:148)
	at org.apache.curator.framework.imps.ExistsBuilderImpl.forPath(ExistsBuilderImpl.java:36)
	at co.solinx.remote.zookeeper.ZookeeperClient.checkExists(ZookeeperClient.java:135)
	at co.solinx.remote.zookeeper.ZookeeperClient.createNote(ZookeeperClient.java:72)
	at co.solinx.forest.registry.zookeeper.ZookeeperRegistry.createRoot(ZookeeperRegistry.java:68)
	at co.solinx.forest.registry.zookeeper.ZookeeperRegistry.registryService(ZookeeperRegistry.java:136)
	at co.solinx.forest.config.ServiceConfig.Export(ServiceConfig.java:54)
	at co.solinx.forest.config.spring.ServiceBean.onApplicationEvent(ServiceBean.java:52)
	at org.springframework.context.event.SimpleApplicationEventMulticaster.invokeListener(SimpleApplicationEventMulticaster.java:151)
	at org.springframework.context.event.SimpleApplicationEventMulticaster.multicastEvent(SimpleApplicationEventMulticaster.java:128)
	at org.springframework.context.support.AbstractApplicationContext.publishEvent(AbstractApplicationContext.java:331)
	at org.springframework.context.support.AbstractApplicationContext.finishRefresh(AbstractApplicationContext.java:773)
	at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:483)
	at org.springframework.context.support.ClassPathXmlApplicationContext.<init>(ClassPathXmlApplicationContext.java:139)
	at org.springframework.context.support.ClassPathXmlApplicationContext.<init>(ClassPathXmlApplicationContext.java:83)
	at co.solinx.forest.container.spring.SpringContainer.start(SpringContainer.java:23)
	at co.solinx.forest.container.Main.main(Main.java:29)
	at provider.main(provider.java:8)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at com.intellij.rt.execution.application.AppMain.main(AppMain.java:140)
2015-05-25 21:14:06,712 [main] ERROR co.solinx.remote.zookeeper.ZookeeperClient (ZookeeperClient.java:87) - org.apache.curator.CuratorConnectionLossException: KeeperErrorCode = ConnectionLoss
2015-05-25 21:14:12,694 [main-SendThread(192.168.140.142:2181)] WARN  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:1102) - Session 0x0 for server null, unexpected error, closing socket connection and attempting reconnect
java.net.ConnectException: Connection timed out: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(Unknown Source)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:361)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1081)
2015-05-25 21:14:13,795 [main-SendThread(192.168.140.142:2181)] INFO  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:975) - Opening socket connection to server 192.168.140.142/192.168.140.142:2181. Will not attempt to authenticate using SASL (unknown error)
2015-05-25 21:14:21,729 [main] WARN  org.apache.curator.ConnectionState (ConnectionState.java:191) - Connection attempt unsuccessful after 63328 (greater than max timeout of 60000). Resetting connection and trying again with a new connection.
2015-05-25 21:14:34,901 [main] INFO  org.apache.zookeeper.ZooKeeper (ZooKeeper.java:684) - Session: 0x0 closed
2015-05-25 21:14:34,901 [main-EventThread] INFO  org.apache.zookeeper.ClientCnxn$EventThread (ClientCnxn.java:512) - EventThread shut down
2015-05-25 21:14:34,902 [main] INFO  org.apache.zookeeper.ZooKeeper (ZooKeeper.java:438) - Initiating client connection, connectString=192.168.140.142:2181 sessionTimeout=60000 watcher=org.apache.curator.ConnectionState@1a38c59b
2015-05-25 21:14:34,908 [main-SendThread(192.168.140.142:2181)] INFO  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:975) - Opening socket connection to server 192.168.140.142/192.168.140.142:2181. Will not attempt to authenticate using SASL (unknown error)
2015-05-25 21:14:55,912 [main-SendThread(192.168.140.142:2181)] WARN  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:1102) - Session 0x0 for server null, unexpected error, closing socket connection and attempting reconnect
java.net.ConnectException: Connection timed out: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(Unknown Source)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:361)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1081)
2015-05-25 21:14:57,014 [main-SendThread(192.168.140.142:2181)] INFO  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:975) - Opening socket connection to server 192.168.140.142/192.168.140.142:2181. Will not attempt to authenticate using SASL (unknown error)
2015-05-25 21:15:12,027 [main] ERROR org.apache.curator.ConnectionState (ConnectionState.java:200) - Connection timed out for connection string (192.168.140.142:2181) and timeout (15000) / elapsed (50298)
org.apache.curator.CuratorConnectionLossException: KeeperErrorCode = ConnectionLoss
	at org.apache.curator.ConnectionState.checkTimeouts(ConnectionState.java:197)
	at org.apache.curator.ConnectionState.getZooKeeper(ConnectionState.java:87)
	at org.apache.curator.CuratorZookeeperClient.getZooKeeper(CuratorZookeeperClient.java:115)
	at org.apache.curator.framework.imps.CuratorFrameworkImpl.getZooKeeper(CuratorFrameworkImpl.java:477)
	at org.apache.curator.framework.imps.ExistsBuilderImpl$2.call(ExistsBuilderImpl.java:172)
	at org.apache.curator.framework.imps.ExistsBuilderImpl$2.call(ExistsBuilderImpl.java:161)
	at org.apache.curator.RetryLoop.callWithRetry(RetryLoop.java:107)
	at org.apache.curator.framework.imps.ExistsBuilderImpl.pathInForeground(ExistsBuilderImpl.java:158)
	at org.apache.curator.framework.imps.ExistsBuilderImpl.forPath(ExistsBuilderImpl.java:148)
	at org.apache.curator.framework.imps.ExistsBuilderImpl.forPath(ExistsBuilderImpl.java:36)
	at co.solinx.remote.zookeeper.ZookeeperClient.checkExists(ZookeeperClient.java:135)
	at co.solinx.remote.zookeeper.ZookeeperClient.createNote(ZookeeperClient.java:72)
	at co.solinx.forest.registry.zookeeper.ZookeeperRegistry.registryServerApi(ZookeeperRegistry.java:78)
	at co.solinx.forest.registry.zookeeper.ZookeeperRegistry.registryService(ZookeeperRegistry.java:137)
	at co.solinx.forest.config.ServiceConfig.Export(ServiceConfig.java:54)
	at co.solinx.forest.config.spring.ServiceBean.onApplicationEvent(ServiceBean.java:52)
	at org.springframework.context.event.SimpleApplicationEventMulticaster.invokeListener(SimpleApplicationEventMulticaster.java:151)
	at org.springframework.context.event.SimpleApplicationEventMulticaster.multicastEvent(SimpleApplicationEventMulticaster.java:128)
	at org.springframework.context.support.AbstractApplicationContext.publishEvent(AbstractApplicationContext.java:331)
	at org.springframework.context.support.AbstractApplicationContext.finishRefresh(AbstractApplicationContext.java:773)
	at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:483)
	at org.springframework.context.support.ClassPathXmlApplicationContext.<init>(ClassPathXmlApplicationContext.java:139)
	at org.springframework.context.support.ClassPathXmlApplicationContext.<init>(ClassPathXmlApplicationContext.java:83)
	at co.solinx.forest.container.spring.SpringContainer.start(SpringContainer.java:23)
	at co.solinx.forest.container.Main.main(Main.java:29)
	at provider.main(provider.java:8)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at com.intellij.rt.execution.application.AppMain.main(AppMain.java:140)
2015-05-25 21:15:18,021 [main-SendThread(192.168.140.142:2181)] WARN  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:1102) - Session 0x0 for server null, unexpected error, closing socket connection and attempting reconnect
java.net.ConnectException: Connection timed out: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(Unknown Source)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:361)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1081)
2015-05-25 21:15:19,123 [main-SendThread(192.168.140.142:2181)] INFO  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:975) - Opening socket connection to server 192.168.140.142/192.168.140.142:2181. Will not attempt to authenticate using SASL (unknown error)
2015-05-25 21:15:28,045 [main] WARN  org.apache.curator.ConnectionState (ConnectionState.java:191) - Connection attempt unsuccessful after 66316 (greater than max timeout of 60000). Resetting connection and trying again with a new connection.
2015-05-25 21:15:40,230 [main] INFO  org.apache.zookeeper.ZooKeeper (ZooKeeper.java:684) - Session: 0x0 closed
2015-05-25 21:15:40,230 [main-EventThread] INFO  org.apache.zookeeper.ClientCnxn$EventThread (ClientCnxn.java:512) - EventThread shut down
2015-05-25 21:15:40,231 [main] INFO  org.apache.zookeeper.ZooKeeper (ZooKeeper.java:438) - Initiating client connection, connectString=192.168.140.142:2181 sessionTimeout=60000 watcher=org.apache.curator.ConnectionState@1a38c59b
2015-05-25 21:15:40,238 [main-SendThread(192.168.140.142:2181)] INFO  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:975) - Opening socket connection to server 192.168.140.142/192.168.140.142:2181. Will not attempt to authenticate using SASL (unknown error)
2015-05-25 21:16:01,244 [main-SendThread(192.168.140.142:2181)] WARN  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:1102) - Session 0x0 for server null, unexpected error, closing socket connection and attempting reconnect
java.net.ConnectException: Connection timed out: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(Unknown Source)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:361)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1081)
2015-05-25 21:16:02,344 [main-SendThread(192.168.140.142:2181)] INFO  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:975) - Opening socket connection to server 192.168.140.142/192.168.140.142:2181. Will not attempt to authenticate using SASL (unknown error)
2015-05-25 21:16:17,357 [main] ERROR org.apache.curator.ConnectionState (ConnectionState.java:200) - Connection timed out for connection string (192.168.140.142:2181) and timeout (15000) / elapsed (49312)
org.apache.curator.CuratorConnectionLossException: KeeperErrorCode = ConnectionLoss
	at org.apache.curator.ConnectionState.checkTimeouts(ConnectionState.java:197)
	at org.apache.curator.ConnectionState.getZooKeeper(ConnectionState.java:87)
	at org.apache.curator.CuratorZookeeperClient.getZooKeeper(CuratorZookeeperClient.java:115)
	at org.apache.curator.framework.imps.CuratorFrameworkImpl.getZooKeeper(CuratorFrameworkImpl.java:477)
	at org.apache.curator.framework.imps.ExistsBuilderImpl$2.call(ExistsBuilderImpl.java:172)
	at org.apache.curator.framework.imps.ExistsBuilderImpl$2.call(ExistsBuilderImpl.java:161)
	at org.apache.curator.RetryLoop.callWithRetry(RetryLoop.java:107)
	at org.apache.curator.framework.imps.ExistsBuilderImpl.pathInForeground(ExistsBuilderImpl.java:158)
	at org.apache.curator.framework.imps.ExistsBuilderImpl.forPath(ExistsBuilderImpl.java:148)
	at org.apache.curator.framework.imps.ExistsBuilderImpl.forPath(ExistsBuilderImpl.java:36)
	at co.solinx.remote.zookeeper.ZookeeperClient.checkExists(ZookeeperClient.java:135)
	at co.solinx.remote.zookeeper.ZookeeperClient.createNote(ZookeeperClient.java:72)
	at co.solinx.forest.registry.zookeeper.ZookeeperRegistry.registryServerApi(ZookeeperRegistry.java:78)
	at co.solinx.forest.registry.zookeeper.ZookeeperRegistry.registryService(ZookeeperRegistry.java:137)
	at co.solinx.forest.config.ServiceConfig.Export(ServiceConfig.java:54)
	at co.solinx.forest.config.spring.ServiceBean.onApplicationEvent(ServiceBean.java:52)
	at org.springframework.context.event.SimpleApplicationEventMulticaster.invokeListener(SimpleApplicationEventMulticaster.java:151)
	at org.springframework.context.event.SimpleApplicationEventMulticaster.multicastEvent(SimpleApplicationEventMulticaster.java:128)
	at org.springframework.context.support.AbstractApplicationContext.publishEvent(AbstractApplicationContext.java:331)
	at org.springframework.context.support.AbstractApplicationContext.finishRefresh(AbstractApplicationContext.java:773)
	at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:483)
	at org.springframework.context.support.ClassPathXmlApplicationContext.<init>(ClassPathXmlApplicationContext.java:139)
	at org.springframework.context.support.ClassPathXmlApplicationContext.<init>(ClassPathXmlApplicationContext.java:83)
	at co.solinx.forest.container.spring.SpringContainer.start(SpringContainer.java:23)
	at co.solinx.forest.container.Main.main(Main.java:29)
	at provider.main(provider.java:8)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at com.intellij.rt.execution.application.AppMain.main(AppMain.java:140)
2015-05-25 21:16:17,358 [main] ERROR co.solinx.remote.zookeeper.ZookeeperClient (ZookeeperClient.java:87) - org.apache.curator.CuratorConnectionLossException: KeeperErrorCode = ConnectionLoss
2015-05-25 21:16:23,349 [main-SendThread(192.168.140.142:2181)] WARN  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:1102) - Session 0x0 for server null, unexpected error, closing socket connection and attempting reconnect
java.net.ConnectException: Connection timed out: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(Unknown Source)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:361)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1081)
2015-05-25 21:16:24,450 [main-SendThread(192.168.140.142:2181)] INFO  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:975) - Opening socket connection to server 192.168.140.142/192.168.140.142:2181. Will not attempt to authenticate using SASL (unknown error)
2015-05-25 21:16:32,375 [main] WARN  org.apache.curator.ConnectionState (ConnectionState.java:191) - Connection attempt unsuccessful after 64330 (greater than max timeout of 60000). Resetting connection and trying again with a new connection.
2015-05-25 21:16:45,557 [main] INFO  org.apache.zookeeper.ZooKeeper (ZooKeeper.java:684) - Session: 0x0 closed
2015-05-25 21:16:45,557 [main-EventThread] INFO  org.apache.zookeeper.ClientCnxn$EventThread (ClientCnxn.java:512) - EventThread shut down
2015-05-25 21:16:45,558 [main] INFO  org.apache.zookeeper.ZooKeeper (ZooKeeper.java:438) - Initiating client connection, connectString=192.168.140.142:2181 sessionTimeout=60000 watcher=org.apache.curator.ConnectionState@1a38c59b
2015-05-25 21:16:45,565 [main-SendThread(192.168.140.142:2181)] INFO  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:975) - Opening socket connection to server 192.168.140.142/192.168.140.142:2181. Will not attempt to authenticate using SASL (unknown error)
2015-05-25 21:17:06,565 [main-SendThread(192.168.140.142:2181)] WARN  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:1102) - Session 0x0 for server null, unexpected error, closing socket connection and attempting reconnect
java.net.ConnectException: Connection timed out: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(Unknown Source)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:361)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1081)
2015-05-25 21:17:07,667 [main-SendThread(192.168.140.142:2181)] INFO  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:975) - Opening socket connection to server 192.168.140.142/192.168.140.142:2181. Will not attempt to authenticate using SASL (unknown error)
2015-05-25 21:17:22,679 [main] ERROR org.apache.curator.ConnectionState (ConnectionState.java:200) - Connection timed out for connection string (192.168.140.142:2181) and timeout (15000) / elapsed (50304)
org.apache.curator.CuratorConnectionLossException: KeeperErrorCode = ConnectionLoss
	at org.apache.curator.ConnectionState.checkTimeouts(ConnectionState.java:197)
	at org.apache.curator.ConnectionState.getZooKeeper(ConnectionState.java:87)
	at org.apache.curator.CuratorZookeeperClient.getZooKeeper(CuratorZookeeperClient.java:115)
	at org.apache.curator.framework.imps.CuratorFrameworkImpl.getZooKeeper(CuratorFrameworkImpl.java:477)
	at org.apache.curator.framework.imps.ExistsBuilderImpl$2.call(ExistsBuilderImpl.java:172)
	at org.apache.curator.framework.imps.ExistsBuilderImpl$2.call(ExistsBuilderImpl.java:161)
	at org.apache.curator.RetryLoop.callWithRetry(RetryLoop.java:107)
	at org.apache.curator.framework.imps.ExistsBuilderImpl.pathInForeground(ExistsBuilderImpl.java:158)
	at org.apache.curator.framework.imps.ExistsBuilderImpl.forPath(ExistsBuilderImpl.java:148)
	at org.apache.curator.framework.imps.ExistsBuilderImpl.forPath(ExistsBuilderImpl.java:36)
	at co.solinx.remote.zookeeper.ZookeeperClient.checkExists(ZookeeperClient.java:135)
	at co.solinx.remote.zookeeper.ZookeeperClient.createNote(ZookeeperClient.java:72)
	at co.solinx.forest.registry.zookeeper.ZookeeperRegistry.createProviderNote(ZookeeperRegistry.java:106)
	at co.solinx.forest.registry.zookeeper.ZookeeperRegistry.registryService(ZookeeperRegistry.java:138)
	at co.solinx.forest.config.ServiceConfig.Export(ServiceConfig.java:54)
	at co.solinx.forest.config.spring.ServiceBean.onApplicationEvent(ServiceBean.java:52)
	at org.springframework.context.event.SimpleApplicationEventMulticaster.invokeListener(SimpleApplicationEventMulticaster.java:151)
	at org.springframework.context.event.SimpleApplicationEventMulticaster.multicastEvent(SimpleApplicationEventMulticaster.java:128)
	at org.springframework.context.support.AbstractApplicationContext.publishEvent(AbstractApplicationContext.java:331)
	at org.springframework.context.support.AbstractApplicationContext.finishRefresh(AbstractApplicationContext.java:773)
	at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:483)
	at org.springframework.context.support.ClassPathXmlApplicationContext.<init>(ClassPathXmlApplicationContext.java:139)
	at org.springframework.context.support.ClassPathXmlApplicationContext.<init>(ClassPathXmlApplicationContext.java:83)
	at co.solinx.forest.container.spring.SpringContainer.start(SpringContainer.java:23)
	at co.solinx.forest.container.Main.main(Main.java:29)
	at provider.main(provider.java:8)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at com.intellij.rt.execution.application.AppMain.main(AppMain.java:140)
2015-05-25 21:17:28,663 [main-SendThread(192.168.140.142:2181)] WARN  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:1102) - Session 0x0 for server null, unexpected error, closing socket connection and attempting reconnect
java.net.ConnectException: Connection timed out: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(Unknown Source)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:361)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1081)
2015-05-25 21:17:29,764 [main-SendThread(192.168.140.142:2181)] INFO  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:975) - Opening socket connection to server 192.168.140.142/192.168.140.142:2181. Will not attempt to authenticate using SASL (unknown error)
2015-05-25 21:17:40,694 [main] WARN  org.apache.curator.ConnectionState (ConnectionState.java:191) - Connection attempt unsuccessful after 68319 (greater than max timeout of 60000). Resetting connection and trying again with a new connection.
2015-05-25 21:17:50,865 [main] INFO  org.apache.zookeeper.ZooKeeper (ZooKeeper.java:684) - Session: 0x0 closed
2015-05-25 21:17:50,865 [main-EventThread] INFO  org.apache.zookeeper.ClientCnxn$EventThread (ClientCnxn.java:512) - EventThread shut down
2015-05-25 21:17:50,866 [main] INFO  org.apache.zookeeper.ZooKeeper (ZooKeeper.java:438) - Initiating client connection, connectString=192.168.140.142:2181 sessionTimeout=60000 watcher=org.apache.curator.ConnectionState@1a38c59b
2015-05-25 21:17:50,872 [main-SendThread(192.168.140.142:2181)] INFO  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:975) - Opening socket connection to server 192.168.140.142/192.168.140.142:2181. Will not attempt to authenticate using SASL (unknown error)
2015-05-25 21:18:11,880 [main-SendThread(192.168.140.142:2181)] WARN  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:1102) - Session 0x0 for server null, unexpected error, closing socket connection and attempting reconnect
java.net.ConnectException: Connection timed out: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(Unknown Source)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:361)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1081)
2015-05-25 21:18:12,982 [main-SendThread(192.168.140.142:2181)] INFO  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:975) - Opening socket connection to server 192.168.140.142/192.168.140.142:2181. Will not attempt to authenticate using SASL (unknown error)
2015-05-25 21:18:30,995 [main] ERROR org.apache.curator.ConnectionState (ConnectionState.java:200) - Connection timed out for connection string (192.168.140.142:2181) and timeout (15000) / elapsed (50300)
org.apache.curator.CuratorConnectionLossException: KeeperErrorCode = ConnectionLoss
	at org.apache.curator.ConnectionState.checkTimeouts(ConnectionState.java:197)
	at org.apache.curator.ConnectionState.getZooKeeper(ConnectionState.java:87)
	at org.apache.curator.CuratorZookeeperClient.getZooKeeper(CuratorZookeeperClient.java:115)
	at org.apache.curator.framework.imps.CuratorFrameworkImpl.getZooKeeper(CuratorFrameworkImpl.java:477)
	at org.apache.curator.framework.imps.ExistsBuilderImpl$2.call(ExistsBuilderImpl.java:172)
	at org.apache.curator.framework.imps.ExistsBuilderImpl$2.call(ExistsBuilderImpl.java:161)
	at org.apache.curator.RetryLoop.callWithRetry(RetryLoop.java:107)
	at org.apache.curator.framework.imps.ExistsBuilderImpl.pathInForeground(ExistsBuilderImpl.java:158)
	at org.apache.curator.framework.imps.ExistsBuilderImpl.forPath(ExistsBuilderImpl.java:148)
	at org.apache.curator.framework.imps.ExistsBuilderImpl.forPath(ExistsBuilderImpl.java:36)
	at co.solinx.remote.zookeeper.ZookeeperClient.checkExists(ZookeeperClient.java:135)
	at co.solinx.remote.zookeeper.ZookeeperClient.createNote(ZookeeperClient.java:72)
	at co.solinx.forest.registry.zookeeper.ZookeeperRegistry.createProviderNote(ZookeeperRegistry.java:106)
	at co.solinx.forest.registry.zookeeper.ZookeeperRegistry.registryService(ZookeeperRegistry.java:138)
	at co.solinx.forest.config.ServiceConfig.Export(ServiceConfig.java:54)
	at co.solinx.forest.config.spring.ServiceBean.onApplicationEvent(ServiceBean.java:52)
	at org.springframework.context.event.SimpleApplicationEventMulticaster.invokeListener(SimpleApplicationEventMulticaster.java:151)
	at org.springframework.context.event.SimpleApplicationEventMulticaster.multicastEvent(SimpleApplicationEventMulticaster.java:128)
	at org.springframework.context.support.AbstractApplicationContext.publishEvent(AbstractApplicationContext.java:331)
	at org.springframework.context.support.AbstractApplicationContext.finishRefresh(AbstractApplicationContext.java:773)
	at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:483)
	at org.springframework.context.support.ClassPathXmlApplicationContext.<init>(ClassPathXmlApplicationContext.java:139)
	at org.springframework.context.support.ClassPathXmlApplicationContext.<init>(ClassPathXmlApplicationContext.java:83)
	at co.solinx.forest.container.spring.SpringContainer.start(SpringContainer.java:23)
	at co.solinx.forest.container.Main.main(Main.java:29)
	at provider.main(provider.java:8)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at com.intellij.rt.execution.application.AppMain.main(AppMain.java:140)
2015-05-25 21:18:30,998 [main] ERROR co.solinx.remote.zookeeper.ZookeeperClient (ZookeeperClient.java:87) - org.apache.curator.CuratorConnectionLossException: KeeperErrorCode = ConnectionLoss
2015-05-25 21:18:33,994 [main-SendThread(192.168.140.142:2181)] WARN  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:1102) - Session 0x0 for server null, unexpected error, closing socket connection and attempting reconnect
java.net.ConnectException: Connection timed out: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(Unknown Source)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:361)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1081)
2015-05-25 21:18:35,094 [main-SendThread(192.168.140.142:2181)] INFO  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:975) - Opening socket connection to server 192.168.140.142/192.168.140.142:2181. Will not attempt to authenticate using SASL (unknown error)
2015-05-25 21:18:46,014 [main] WARN  org.apache.curator.ConnectionState (ConnectionState.java:191) - Connection attempt unsuccessful after 65319 (greater than max timeout of 60000). Resetting connection and trying again with a new connection.
2015-05-25 21:18:56,204 [main] INFO  org.apache.zookeeper.ZooKeeper (ZooKeeper.java:684) - Session: 0x0 closed
2015-05-25 21:18:56,204 [main-EventThread] INFO  org.apache.zookeeper.ClientCnxn$EventThread (ClientCnxn.java:512) - EventThread shut down
2015-05-25 21:18:56,204 [main] INFO  org.apache.zookeeper.ZooKeeper (ZooKeeper.java:438) - Initiating client connection, connectString=192.168.140.142:2181 sessionTimeout=60000 watcher=org.apache.curator.ConnectionState@1a38c59b
2015-05-25 21:18:56,207 [main-SendThread(192.168.140.142:2181)] INFO  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:975) - Opening socket connection to server 192.168.140.142/192.168.140.142:2181. Will not attempt to authenticate using SASL (unknown error)
2015-05-25 21:19:17,207 [main-SendThread(192.168.140.142:2181)] WARN  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:1102) - Session 0x0 for server null, unexpected error, closing socket connection and attempting reconnect
java.net.ConnectException: Connection timed out: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(Unknown Source)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:361)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1081)
2015-05-25 21:19:18,308 [main-SendThread(192.168.140.142:2181)] INFO  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:975) - Opening socket connection to server 192.168.140.142/192.168.140.142:2181. Will not attempt to authenticate using SASL (unknown error)
2015-05-25 21:19:33,323 [main] ERROR org.apache.curator.ConnectionState (ConnectionState.java:200) - Connection timed out for connection string (192.168.140.142:2181) and timeout (15000) / elapsed (47309)
org.apache.curator.CuratorConnectionLossException: KeeperErrorCode = ConnectionLoss
	at org.apache.curator.ConnectionState.checkTimeouts(ConnectionState.java:197)
	at org.apache.curator.ConnectionState.getZooKeeper(ConnectionState.java:87)
	at org.apache.curator.CuratorZookeeperClient.getZooKeeper(CuratorZookeeperClient.java:115)
	at org.apache.curator.framework.imps.CuratorFrameworkImpl.getZooKeeper(CuratorFrameworkImpl.java:477)
	at org.apache.curator.framework.imps.ExistsBuilderImpl$2.call(ExistsBuilderImpl.java:172)
	at org.apache.curator.framework.imps.ExistsBuilderImpl$2.call(ExistsBuilderImpl.java:161)
	at org.apache.curator.RetryLoop.callWithRetry(RetryLoop.java:107)
	at org.apache.curator.framework.imps.ExistsBuilderImpl.pathInForeground(ExistsBuilderImpl.java:158)
	at org.apache.curator.framework.imps.ExistsBuilderImpl.forPath(ExistsBuilderImpl.java:148)
	at org.apache.curator.framework.imps.ExistsBuilderImpl.forPath(ExistsBuilderImpl.java:36)
	at co.solinx.remote.zookeeper.ZookeeperClient.checkExists(ZookeeperClient.java:135)
	at co.solinx.remote.zookeeper.ZookeeperClient.createNote(ZookeeperClient.java:72)
	at co.solinx.forest.registry.zookeeper.ZookeeperRegistry.registryService(ZookeeperRegistry.java:146)
	at co.solinx.forest.config.ServiceConfig.Export(ServiceConfig.java:54)
	at co.solinx.forest.config.spring.ServiceBean.onApplicationEvent(ServiceBean.java:52)
	at org.springframework.context.event.SimpleApplicationEventMulticaster.invokeListener(SimpleApplicationEventMulticaster.java:151)
	at org.springframework.context.event.SimpleApplicationEventMulticaster.multicastEvent(SimpleApplicationEventMulticaster.java:128)
	at org.springframework.context.support.AbstractApplicationContext.publishEvent(AbstractApplicationContext.java:331)
	at org.springframework.context.support.AbstractApplicationContext.finishRefresh(AbstractApplicationContext.java:773)
	at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:483)
	at org.springframework.context.support.ClassPathXmlApplicationContext.<init>(ClassPathXmlApplicationContext.java:139)
	at org.springframework.context.support.ClassPathXmlApplicationContext.<init>(ClassPathXmlApplicationContext.java:83)
	at co.solinx.forest.container.spring.SpringContainer.start(SpringContainer.java:23)
	at co.solinx.forest.container.Main.main(Main.java:29)
	at provider.main(provider.java:8)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at com.intellij.rt.execution.application.AppMain.main(AppMain.java:140)
2015-05-25 21:19:39,310 [main-SendThread(192.168.140.142:2181)] WARN  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:1102) - Session 0x0 for server null, unexpected error, closing socket connection and attempting reconnect
java.net.ConnectException: Connection timed out: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(Unknown Source)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:361)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1081)
2015-05-25 21:19:40,411 [main-SendThread(192.168.140.142:2181)] INFO  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:975) - Opening socket connection to server 192.168.140.142/192.168.140.142:2181. Will not attempt to authenticate using SASL (unknown error)
2015-05-25 21:19:49,338 [main] WARN  org.apache.curator.ConnectionState (ConnectionState.java:191) - Connection attempt unsuccessful after 63324 (greater than max timeout of 60000). Resetting connection and trying again with a new connection.
2015-05-25 21:20:01,532 [main-EventThread] INFO  org.apache.zookeeper.ClientCnxn$EventThread (ClientCnxn.java:512) - EventThread shut down
2015-05-25 21:20:01,532 [main] INFO  org.apache.zookeeper.ZooKeeper (ZooKeeper.java:684) - Session: 0x0 closed
2015-05-25 21:20:01,533 [main] INFO  org.apache.zookeeper.ZooKeeper (ZooKeeper.java:438) - Initiating client connection, connectString=192.168.140.142:2181 sessionTimeout=60000 watcher=org.apache.curator.ConnectionState@1a38c59b
2015-05-25 21:20:01,536 [main-SendThread(192.168.140.142:2181)] INFO  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:975) - Opening socket connection to server 192.168.140.142/192.168.140.142:2181. Will not attempt to authenticate using SASL (unknown error)
2015-05-25 21:20:22,538 [main-SendThread(192.168.140.142:2181)] WARN  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:1102) - Session 0x0 for server null, unexpected error, closing socket connection and attempting reconnect
java.net.ConnectException: Connection timed out: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(Unknown Source)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:361)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1081)
2015-05-25 21:20:23,638 [main-SendThread(192.168.140.142:2181)] INFO  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:975) - Opening socket connection to server 192.168.140.142/192.168.140.142:2181. Will not attempt to authenticate using SASL (unknown error)
2015-05-25 21:20:38,651 [main] ERROR org.apache.curator.ConnectionState (ConnectionState.java:200) - Connection timed out for connection string (192.168.140.142:2181) and timeout (15000) / elapsed (49312)
org.apache.curator.CuratorConnectionLossException: KeeperErrorCode = ConnectionLoss
	at org.apache.curator.ConnectionState.checkTimeouts(ConnectionState.java:197)
	at org.apache.curator.ConnectionState.getZooKeeper(ConnectionState.java:87)
	at org.apache.curator.CuratorZookeeperClient.getZooKeeper(CuratorZookeeperClient.java:115)
	at org.apache.curator.framework.imps.CuratorFrameworkImpl.getZooKeeper(CuratorFrameworkImpl.java:477)
	at org.apache.curator.framework.imps.ExistsBuilderImpl$2.call(ExistsBuilderImpl.java:172)
	at org.apache.curator.framework.imps.ExistsBuilderImpl$2.call(ExistsBuilderImpl.java:161)
	at org.apache.curator.RetryLoop.callWithRetry(RetryLoop.java:107)
	at org.apache.curator.framework.imps.ExistsBuilderImpl.pathInForeground(ExistsBuilderImpl.java:158)
	at org.apache.curator.framework.imps.ExistsBuilderImpl.forPath(ExistsBuilderImpl.java:148)
	at org.apache.curator.framework.imps.ExistsBuilderImpl.forPath(ExistsBuilderImpl.java:36)
	at co.solinx.remote.zookeeper.ZookeeperClient.checkExists(ZookeeperClient.java:135)
	at co.solinx.remote.zookeeper.ZookeeperClient.createNote(ZookeeperClient.java:72)
	at co.solinx.forest.registry.zookeeper.ZookeeperRegistry.registryService(ZookeeperRegistry.java:146)
	at co.solinx.forest.config.ServiceConfig.Export(ServiceConfig.java:54)
	at co.solinx.forest.config.spring.ServiceBean.onApplicationEvent(ServiceBean.java:52)
	at org.springframework.context.event.SimpleApplicationEventMulticaster.invokeListener(SimpleApplicationEventMulticaster.java:151)
	at org.springframework.context.event.SimpleApplicationEventMulticaster.multicastEvent(SimpleApplicationEventMulticaster.java:128)
	at org.springframework.context.support.AbstractApplicationContext.publishEvent(AbstractApplicationContext.java:331)
	at org.springframework.context.support.AbstractApplicationContext.finishRefresh(AbstractApplicationContext.java:773)
	at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:483)
	at org.springframework.context.support.ClassPathXmlApplicationContext.<init>(ClassPathXmlApplicationContext.java:139)
	at org.springframework.context.support.ClassPathXmlApplicationContext.<init>(ClassPathXmlApplicationContext.java:83)
	at co.solinx.forest.container.spring.SpringContainer.start(SpringContainer.java:23)
	at co.solinx.forest.container.Main.main(Main.java:29)
	at provider.main(provider.java:8)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at com.intellij.rt.execution.application.AppMain.main(AppMain.java:140)
2015-05-25 21:20:38,652 [main] ERROR co.solinx.remote.zookeeper.ZookeeperClient (ZookeeperClient.java:87) - org.apache.curator.CuratorConnectionLossException: KeeperErrorCode = ConnectionLoss
2015-05-25 21:20:38,652 [main] INFO  co.solinx.forest.config.ServiceConfig (ServiceConfig.java:55) - [co.solinx.forest.demo.impl.HelloForestServiceImpl@4b5d6a01, co.solinx.forest.demo.impl.HelloRpcServiceImpl@703580bf]
2015-05-25 21:20:40,153 [main] INFO  solinx.forest.remote.netty.server.NettyServer (NettyServer.java:65) - netty server start bind by 18088
2015-05-25 21:20:40,153 [main] INFO  solinx.forest.remote.netty.server.NettyServer (NettyServer.java:48) - ------------------netty start
2015-05-25 21:20:44,643 [main-SendThread(192.168.140.142:2181)] WARN  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:1102) - Session 0x0 for server null, unexpected error, closing socket connection and attempting reconnect
java.net.ConnectException: Connection timed out: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(Unknown Source)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:361)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1081)
2015-05-25 21:20:45,745 [main-SendThread(192.168.140.142:2181)] INFO  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:975) - Opening socket connection to server 192.168.140.142/192.168.140.142:2181. Will not attempt to authenticate using SASL (unknown error)
2015-05-25 21:21:06,748 [main-SendThread(192.168.140.142:2181)] WARN  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:1102) - Session 0x0 for server null, unexpected error, closing socket connection and attempting reconnect
java.net.ConnectException: Connection timed out: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(Unknown Source)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:361)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1081)
2015-05-25 21:21:07,850 [main-SendThread(192.168.140.142:2181)] INFO  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:975) - Opening socket connection to server 192.168.140.142/192.168.140.142:2181. Will not attempt to authenticate using SASL (unknown error)
2015-05-25 21:21:28,854 [main-SendThread(192.168.140.142:2181)] WARN  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:1102) - Session 0x0 for server null, unexpected error, closing socket connection and attempting reconnect
java.net.ConnectException: Connection timed out: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(Unknown Source)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:361)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1081)
2015-05-25 21:21:29,956 [main-SendThread(192.168.140.142:2181)] INFO  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:975) - Opening socket connection to server 192.168.140.142/192.168.140.142:2181. Will not attempt to authenticate using SASL (unknown error)
2015-05-25 21:21:50,960 [main-SendThread(192.168.140.142:2181)] WARN  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:1102) - Session 0x0 for server null, unexpected error, closing socket connection and attempting reconnect
java.net.ConnectException: Connection timed out: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(Unknown Source)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:361)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1081)
2015-05-25 21:21:52,062 [main-SendThread(192.168.140.142:2181)] INFO  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:975) - Opening socket connection to server 192.168.140.142/192.168.140.142:2181. Will not attempt to authenticate using SASL (unknown error)
2015-05-25 21:22:13,063 [main-SendThread(192.168.140.142:2181)] WARN  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:1102) - Session 0x0 for server null, unexpected error, closing socket connection and attempting reconnect
java.net.ConnectException: Connection timed out: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(Unknown Source)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:361)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1081)
2015-05-25 21:22:14,163 [main-SendThread(192.168.140.142:2181)] INFO  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:975) - Opening socket connection to server 192.168.140.142/192.168.140.142:2181. Will not attempt to authenticate using SASL (unknown error)
2015-05-25 21:22:35,163 [main-SendThread(192.168.140.142:2181)] WARN  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:1102) - Session 0x0 for server null, unexpected error, closing socket connection and attempting reconnect
java.net.ConnectException: Connection timed out: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(Unknown Source)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:361)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1081)
2015-05-25 21:22:36,264 [main-SendThread(192.168.140.142:2181)] INFO  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:975) - Opening socket connection to server 192.168.140.142/192.168.140.142:2181. Will not attempt to authenticate using SASL (unknown error)
2015-05-25 21:22:57,269 [main-SendThread(192.168.140.142:2181)] WARN  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:1102) - Session 0x0 for server null, unexpected error, closing socket connection and attempting reconnect
java.net.ConnectException: Connection timed out: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(Unknown Source)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:361)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1081)
2015-05-25 21:22:58,371 [main-SendThread(192.168.140.142:2181)] INFO  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:975) - Opening socket connection to server 192.168.140.142/192.168.140.142:2181. Will not attempt to authenticate using SASL (unknown error)
2015-05-25 21:23:19,382 [main-SendThread(192.168.140.142:2181)] WARN  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:1102) - Session 0x0 for server null, unexpected error, closing socket connection and attempting reconnect
java.net.ConnectException: Connection timed out: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(Unknown Source)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:361)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1081)
2015-05-25 21:23:20,482 [main-SendThread(192.168.140.142:2181)] INFO  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:975) - Opening socket connection to server 192.168.140.142/192.168.140.142:2181. Will not attempt to authenticate using SASL (unknown error)
2015-05-25 21:23:41,487 [main-SendThread(192.168.140.142:2181)] WARN  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:1102) - Session 0x0 for server null, unexpected error, closing socket connection and attempting reconnect
java.net.ConnectException: Connection timed out: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(Unknown Source)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:361)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1081)
2015-05-25 21:23:42,589 [main-SendThread(192.168.140.142:2181)] INFO  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:975) - Opening socket connection to server 192.168.140.142/192.168.140.142:2181. Will not attempt to authenticate using SASL (unknown error)
2015-05-25 21:24:03,589 [main-SendThread(192.168.140.142:2181)] WARN  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:1102) - Session 0x0 for server null, unexpected error, closing socket connection and attempting reconnect
java.net.ConnectException: Connection timed out: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(Unknown Source)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:361)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1081)
2015-05-25 21:24:04,690 [main-SendThread(192.168.140.142:2181)] INFO  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:975) - Opening socket connection to server 192.168.140.142/192.168.140.142:2181. Will not attempt to authenticate using SASL (unknown error)
2015-05-25 21:24:25,698 [main-SendThread(192.168.140.142:2181)] WARN  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:1102) - Session 0x0 for server null, unexpected error, closing socket connection and attempting reconnect
java.net.ConnectException: Connection timed out: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(Unknown Source)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:361)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1081)
2015-05-25 21:24:26,800 [main-SendThread(192.168.140.142:2181)] INFO  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:975) - Opening socket connection to server 192.168.140.142/192.168.140.142:2181. Will not attempt to authenticate using SASL (unknown error)
2015-05-25 21:24:47,811 [main-SendThread(192.168.140.142:2181)] WARN  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:1102) - Session 0x0 for server null, unexpected error, closing socket connection and attempting reconnect
java.net.ConnectException: Connection timed out: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(Unknown Source)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:361)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1081)
2015-05-25 21:24:48,912 [main-SendThread(192.168.140.142:2181)] INFO  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:975) - Opening socket connection to server 192.168.140.142/192.168.140.142:2181. Will not attempt to authenticate using SASL (unknown error)
2015-05-25 21:25:09,913 [main-SendThread(192.168.140.142:2181)] WARN  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:1102) - Session 0x0 for server null, unexpected error, closing socket connection and attempting reconnect
java.net.ConnectException: Connection timed out: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(Unknown Source)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:361)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1081)
2015-05-25 21:25:11,013 [main-SendThread(192.168.140.142:2181)] INFO  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:975) - Opening socket connection to server 192.168.140.142/192.168.140.142:2181. Will not attempt to authenticate using SASL (unknown error)
2015-05-25 21:25:32,018 [main-SendThread(192.168.140.142:2181)] WARN  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:1102) - Session 0x0 for server null, unexpected error, closing socket connection and attempting reconnect
java.net.ConnectException: Connection timed out: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(Unknown Source)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:361)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1081)
2015-05-25 21:25:33,120 [main-SendThread(192.168.140.142:2181)] INFO  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:975) - Opening socket connection to server 192.168.140.142/192.168.140.142:2181. Will not attempt to authenticate using SASL (unknown error)
2015-05-25 21:25:54,116 [main-SendThread(192.168.140.142:2181)] WARN  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:1102) - Session 0x0 for server null, unexpected error, closing socket connection and attempting reconnect
java.net.ConnectException: Connection timed out: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(Unknown Source)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:361)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1081)
2015-05-25 21:25:55,218 [main-SendThread(192.168.140.142:2181)] INFO  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:975) - Opening socket connection to server 192.168.140.142/192.168.140.142:2181. Will not attempt to authenticate using SASL (unknown error)
2015-05-25 21:26:16,221 [main-SendThread(192.168.140.142:2181)] WARN  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:1102) - Session 0x0 for server null, unexpected error, closing socket connection and attempting reconnect
java.net.ConnectException: Connection timed out: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(Unknown Source)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:361)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1081)
2015-05-25 21:26:17,323 [main-SendThread(192.168.140.142:2181)] INFO  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:975) - Opening socket connection to server 192.168.140.142/192.168.140.142:2181. Will not attempt to authenticate using SASL (unknown error)
2015-05-25 21:26:38,330 [main-SendThread(192.168.140.142:2181)] WARN  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:1102) - Session 0x0 for server null, unexpected error, closing socket connection and attempting reconnect
java.net.ConnectException: Connection timed out: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(Unknown Source)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:361)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1081)
2015-05-25 21:26:39,432 [main-SendThread(192.168.140.142:2181)] INFO  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:975) - Opening socket connection to server 192.168.140.142/192.168.140.142:2181. Will not attempt to authenticate using SASL (unknown error)
2015-05-25 21:27:00,433 [main-SendThread(192.168.140.142:2181)] WARN  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:1102) - Session 0x0 for server null, unexpected error, closing socket connection and attempting reconnect
java.net.ConnectException: Connection timed out: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(Unknown Source)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:361)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1081)
2015-05-25 21:27:01,534 [main-SendThread(192.168.140.142:2181)] INFO  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:975) - Opening socket connection to server 192.168.140.142/192.168.140.142:2181. Will not attempt to authenticate using SASL (unknown error)
2015-05-25 21:27:22,535 [main-SendThread(192.168.140.142:2181)] WARN  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:1102) - Session 0x0 for server null, unexpected error, closing socket connection and attempting reconnect
java.net.ConnectException: Connection timed out: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(Unknown Source)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:361)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1081)
2015-05-25 21:27:23,636 [main-SendThread(192.168.140.142:2181)] INFO  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:975) - Opening socket connection to server 192.168.140.142/192.168.140.142:2181. Will not attempt to authenticate using SASL (unknown error)
2015-05-25 21:27:44,648 [main-SendThread(192.168.140.142:2181)] WARN  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:1102) - Session 0x0 for server null, unexpected error, closing socket connection and attempting reconnect
java.net.ConnectException: Connection timed out: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(Unknown Source)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:361)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1081)
2015-05-25 21:27:45,750 [main-SendThread(192.168.140.142:2181)] INFO  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:975) - Opening socket connection to server 192.168.140.142/192.168.140.142:2181. Will not attempt to authenticate using SASL (unknown error)
2015-05-25 21:28:06,742 [main-SendThread(192.168.140.142:2181)] WARN  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:1102) - Session 0x0 for server null, unexpected error, closing socket connection and attempting reconnect
java.net.ConnectException: Connection timed out: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(Unknown Source)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:361)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1081)
2015-05-25 21:28:07,843 [main-SendThread(192.168.140.142:2181)] INFO  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:975) - Opening socket connection to server 192.168.140.142/192.168.140.142:2181. Will not attempt to authenticate using SASL (unknown error)
2015-05-25 21:28:28,849 [main-SendThread(192.168.140.142:2181)] WARN  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:1102) - Session 0x0 for server null, unexpected error, closing socket connection and attempting reconnect
java.net.ConnectException: Connection timed out: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(Unknown Source)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:361)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1081)
2015-05-25 21:28:29,951 [main-SendThread(192.168.140.142:2181)] INFO  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:975) - Opening socket connection to server 192.168.140.142/192.168.140.142:2181. Will not attempt to authenticate using SASL (unknown error)
2015-05-25 21:28:50,953 [main-SendThread(192.168.140.142:2181)] WARN  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:1102) - Session 0x0 for server null, unexpected error, closing socket connection and attempting reconnect
java.net.ConnectException: Connection timed out: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(Unknown Source)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:361)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1081)
2015-05-25 21:28:52,054 [main-SendThread(192.168.140.142:2181)] INFO  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:975) - Opening socket connection to server 192.168.140.142/192.168.140.142:2181. Will not attempt to authenticate using SASL (unknown error)
2015-05-25 21:29:13,057 [main-SendThread(192.168.140.142:2181)] WARN  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:1102) - Session 0x0 for server null, unexpected error, closing socket connection and attempting reconnect
java.net.ConnectException: Connection timed out: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(Unknown Source)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:361)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1081)
2015-05-25 21:29:14,157 [main-SendThread(192.168.140.142:2181)] INFO  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:975) - Opening socket connection to server 192.168.140.142/192.168.140.142:2181. Will not attempt to authenticate using SASL (unknown error)
2015-05-25 21:29:35,154 [main-SendThread(192.168.140.142:2181)] WARN  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:1102) - Session 0x0 for server null, unexpected error, closing socket connection and attempting reconnect
java.net.ConnectException: Connection timed out: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(Unknown Source)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:361)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1081)
2015-05-25 21:29:36,256 [main-SendThread(192.168.140.142:2181)] INFO  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:975) - Opening socket connection to server 192.168.140.142/192.168.140.142:2181. Will not attempt to authenticate using SASL (unknown error)
2015-05-25 21:29:57,261 [main-SendThread(192.168.140.142:2181)] WARN  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:1102) - Session 0x0 for server null, unexpected error, closing socket connection and attempting reconnect
java.net.ConnectException: Connection timed out: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(Unknown Source)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:361)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1081)
2015-05-25 21:29:58,363 [main-SendThread(192.168.140.142:2181)] INFO  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:975) - Opening socket connection to server 192.168.140.142/192.168.140.142:2181. Will not attempt to authenticate using SASL (unknown error)
2015-05-25 21:30:19,366 [main-SendThread(192.168.140.142:2181)] WARN  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:1102) - Session 0x0 for server null, unexpected error, closing socket connection and attempting reconnect
java.net.ConnectException: Connection timed out: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(Unknown Source)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:361)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1081)
2015-05-25 21:30:20,468 [main-SendThread(192.168.140.142:2181)] INFO  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:975) - Opening socket connection to server 192.168.140.142/192.168.140.142:2181. Will not attempt to authenticate using SASL (unknown error)
2015-05-25 21:30:41,478 [main-SendThread(192.168.140.142:2181)] WARN  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:1102) - Session 0x0 for server null, unexpected error, closing socket connection and attempting reconnect
java.net.ConnectException: Connection timed out: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(Unknown Source)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:361)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1081)
2015-05-25 21:30:42,580 [main-SendThread(192.168.140.142:2181)] INFO  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:975) - Opening socket connection to server 192.168.140.142/192.168.140.142:2181. Will not attempt to authenticate using SASL (unknown error)
2015-05-25 21:31:03,590 [main-SendThread(192.168.140.142:2181)] WARN  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:1102) - Session 0x0 for server null, unexpected error, closing socket connection and attempting reconnect
java.net.ConnectException: Connection timed out: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(Unknown Source)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:361)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1081)
2015-05-25 21:31:04,691 [main-SendThread(192.168.140.142:2181)] INFO  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:975) - Opening socket connection to server 192.168.140.142/192.168.140.142:2181. Will not attempt to authenticate using SASL (unknown error)
2015-05-25 21:31:25,697 [main-SendThread(192.168.140.142:2181)] WARN  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:1102) - Session 0x0 for server null, unexpected error, closing socket connection and attempting reconnect
java.net.ConnectException: Connection timed out: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(Unknown Source)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:361)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1081)
2015-05-25 21:31:26,799 [main-SendThread(192.168.140.142:2181)] INFO  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:975) - Opening socket connection to server 192.168.140.142/192.168.140.142:2181. Will not attempt to authenticate using SASL (unknown error)
2015-05-25 21:31:47,814 [main-SendThread(192.168.140.142:2181)] WARN  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:1102) - Session 0x0 for server null, unexpected error, closing socket connection and attempting reconnect
java.net.ConnectException: Connection timed out: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(Unknown Source)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:361)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1081)
2015-05-25 21:31:48,916 [main-SendThread(192.168.140.142:2181)] INFO  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:975) - Opening socket connection to server 192.168.140.142/192.168.140.142:2181. Will not attempt to authenticate using SASL (unknown error)
2015-05-25 21:32:09,920 [main-SendThread(192.168.140.142:2181)] WARN  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:1102) - Session 0x0 for server null, unexpected error, closing socket connection and attempting reconnect
java.net.ConnectException: Connection timed out: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(Unknown Source)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:361)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1081)
2015-05-25 21:32:11,022 [main-SendThread(192.168.140.142:2181)] INFO  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:975) - Opening socket connection to server 192.168.140.142/192.168.140.142:2181. Will not attempt to authenticate using SASL (unknown error)
2015-05-25 21:32:32,031 [main-SendThread(192.168.140.142:2181)] WARN  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:1102) - Session 0x0 for server null, unexpected error, closing socket connection and attempting reconnect
java.net.ConnectException: Connection timed out: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(Unknown Source)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:361)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1081)
2015-05-25 21:32:33,132 [main-SendThread(192.168.140.142:2181)] INFO  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:975) - Opening socket connection to server 192.168.140.142/192.168.140.142:2181. Will not attempt to authenticate using SASL (unknown error)
2015-05-25 21:32:54,137 [main-SendThread(192.168.140.142:2181)] WARN  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:1102) - Session 0x0 for server null, unexpected error, closing socket connection and attempting reconnect
java.net.ConnectException: Connection timed out: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(Unknown Source)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:361)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1081)
2015-05-25 21:32:55,238 [main-SendThread(192.168.140.142:2181)] INFO  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:975) - Opening socket connection to server 192.168.140.142/192.168.140.142:2181. Will not attempt to authenticate using SASL (unknown error)
2015-05-25 21:33:16,241 [main-SendThread(192.168.140.142:2181)] WARN  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:1102) - Session 0x0 for server null, unexpected error, closing socket connection and attempting reconnect
java.net.ConnectException: Connection timed out: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(Unknown Source)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:361)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1081)
2015-05-25 21:33:17,343 [main-SendThread(192.168.140.142:2181)] INFO  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:975) - Opening socket connection to server 192.168.140.142/192.168.140.142:2181. Will not attempt to authenticate using SASL (unknown error)
2015-05-25 21:33:38,352 [main-SendThread(192.168.140.142:2181)] WARN  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:1102) - Session 0x0 for server null, unexpected error, closing socket connection and attempting reconnect
java.net.ConnectException: Connection timed out: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(Unknown Source)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:361)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1081)
2015-05-25 21:33:39,452 [main-SendThread(192.168.140.142:2181)] INFO  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:975) - Opening socket connection to server 192.168.140.142/192.168.140.142:2181. Will not attempt to authenticate using SASL (unknown error)
2015-05-25 21:34:00,460 [main-SendThread(192.168.140.142:2181)] WARN  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:1102) - Session 0x0 for server null, unexpected error, closing socket connection and attempting reconnect
java.net.ConnectException: Connection timed out: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(Unknown Source)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:361)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1081)
2015-05-25 21:34:01,561 [main-SendThread(192.168.140.142:2181)] INFO  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:975) - Opening socket connection to server 192.168.140.142/192.168.140.142:2181. Will not attempt to authenticate using SASL (unknown error)
2015-05-25 21:34:22,568 [main-SendThread(192.168.140.142:2181)] WARN  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:1102) - Session 0x0 for server null, unexpected error, closing socket connection and attempting reconnect
java.net.ConnectException: Connection timed out: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(Unknown Source)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:361)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1081)
2015-05-25 21:34:23,670 [main-SendThread(192.168.140.142:2181)] INFO  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:975) - Opening socket connection to server 192.168.140.142/192.168.140.142:2181. Will not attempt to authenticate using SASL (unknown error)
2015-05-25 21:34:44,676 [main-SendThread(192.168.140.142:2181)] WARN  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:1102) - Session 0x0 for server null, unexpected error, closing socket connection and attempting reconnect
java.net.ConnectException: Connection timed out: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(Unknown Source)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:361)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1081)
2015-05-25 21:34:45,778 [main-SendThread(192.168.140.142:2181)] INFO  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:975) - Opening socket connection to server 192.168.140.142/192.168.140.142:2181. Will not attempt to authenticate using SASL (unknown error)
2015-05-25 21:35:06,782 [main-SendThread(192.168.140.142:2181)] WARN  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:1102) - Session 0x0 for server null, unexpected error, closing socket connection and attempting reconnect
java.net.ConnectException: Connection timed out: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(Unknown Source)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:361)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1081)
2015-05-25 21:35:07,884 [main-SendThread(192.168.140.142:2181)] INFO  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:975) - Opening socket connection to server 192.168.140.142/192.168.140.142:2181. Will not attempt to authenticate using SASL (unknown error)
2015-05-25 21:35:28,884 [main-SendThread(192.168.140.142:2181)] WARN  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:1102) - Session 0x0 for server null, unexpected error, closing socket connection and attempting reconnect
java.net.ConnectException: Connection timed out: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(Unknown Source)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:361)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1081)
2015-05-25 21:35:29,986 [main-SendThread(192.168.140.142:2181)] INFO  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:975) - Opening socket connection to server 192.168.140.142/192.168.140.142:2181. Will not attempt to authenticate using SASL (unknown error)
2015-05-25 21:35:50,988 [main-SendThread(192.168.140.142:2181)] WARN  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:1102) - Session 0x0 for server null, unexpected error, closing socket connection and attempting reconnect
java.net.ConnectException: Connection timed out: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(Unknown Source)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:361)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1081)
2015-05-25 21:35:52,090 [main-SendThread(192.168.140.142:2181)] INFO  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:975) - Opening socket connection to server 192.168.140.142/192.168.140.142:2181. Will not attempt to authenticate using SASL (unknown error)
2015-05-25 21:36:13,094 [main-SendThread(192.168.140.142:2181)] WARN  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:1102) - Session 0x0 for server null, unexpected error, closing socket connection and attempting reconnect
java.net.ConnectException: Connection timed out: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(Unknown Source)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:361)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1081)
2015-05-25 21:36:14,196 [main-SendThread(192.168.140.142:2181)] INFO  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:975) - Opening socket connection to server 192.168.140.142/192.168.140.142:2181. Will not attempt to authenticate using SASL (unknown error)
2015-05-25 21:36:35,192 [main-SendThread(192.168.140.142:2181)] WARN  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:1102) - Session 0x0 for server null, unexpected error, closing socket connection and attempting reconnect
java.net.ConnectException: Connection timed out: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(Unknown Source)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:361)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1081)
2015-05-25 21:36:36,294 [main-SendThread(192.168.140.142:2181)] INFO  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:975) - Opening socket connection to server 192.168.140.142/192.168.140.142:2181. Will not attempt to authenticate using SASL (unknown error)
2015-05-25 21:36:57,293 [main-SendThread(192.168.140.142:2181)] WARN  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:1102) - Session 0x0 for server null, unexpected error, closing socket connection and attempting reconnect
java.net.ConnectException: Connection timed out: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(Unknown Source)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:361)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1081)
2015-05-25 21:36:58,394 [main-SendThread(192.168.140.142:2181)] INFO  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:975) - Opening socket connection to server 192.168.140.142/192.168.140.142:2181. Will not attempt to authenticate using SASL (unknown error)
2015-05-25 21:37:19,394 [main-SendThread(192.168.140.142:2181)] WARN  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:1102) - Session 0x0 for server null, unexpected error, closing socket connection and attempting reconnect
java.net.ConnectException: Connection timed out: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(Unknown Source)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:361)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1081)
2015-05-25 21:37:20,495 [main-SendThread(192.168.140.142:2181)] INFO  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:975) - Opening socket connection to server 192.168.140.142/192.168.140.142:2181. Will not attempt to authenticate using SASL (unknown error)
2015-05-25 21:37:41,492 [main-SendThread(192.168.140.142:2181)] WARN  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:1102) - Session 0x0 for server null, unexpected error, closing socket connection and attempting reconnect
java.net.ConnectException: Connection timed out: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(Unknown Source)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:361)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1081)
2015-05-25 21:37:42,594 [main-SendThread(192.168.140.142:2181)] INFO  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:975) - Opening socket connection to server 192.168.140.142/192.168.140.142:2181. Will not attempt to authenticate using SASL (unknown error)
2015-05-25 21:38:03,612 [main-SendThread(192.168.140.142:2181)] WARN  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:1102) - Session 0x0 for server null, unexpected error, closing socket connection and attempting reconnect
java.net.ConnectException: Connection timed out: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(Unknown Source)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:361)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1081)
2015-05-25 21:38:04,712 [main-SendThread(192.168.140.142:2181)] INFO  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:975) - Opening socket connection to server 192.168.140.142/192.168.140.142:2181. Will not attempt to authenticate using SASL (unknown error)
2015-05-25 21:38:25,718 [main-SendThread(192.168.140.142:2181)] WARN  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:1102) - Session 0x0 for server null, unexpected error, closing socket connection and attempting reconnect
java.net.ConnectException: Connection timed out: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(Unknown Source)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:361)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1081)
2015-05-25 21:38:26,819 [main-SendThread(192.168.140.142:2181)] INFO  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:975) - Opening socket connection to server 192.168.140.142/192.168.140.142:2181. Will not attempt to authenticate using SASL (unknown error)
2015-05-25 21:38:47,824 [main-SendThread(192.168.140.142:2181)] WARN  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:1102) - Session 0x0 for server null, unexpected error, closing socket connection and attempting reconnect
java.net.ConnectException: Connection timed out: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(Unknown Source)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:361)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1081)
2015-05-25 21:38:48,926 [main-SendThread(192.168.140.142:2181)] INFO  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:975) - Opening socket connection to server 192.168.140.142/192.168.140.142:2181. Will not attempt to authenticate using SASL (unknown error)
2015-05-25 21:39:09,926 [main-SendThread(192.168.140.142:2181)] WARN  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:1102) - Session 0x0 for server null, unexpected error, closing socket connection and attempting reconnect
java.net.ConnectException: Connection timed out: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(Unknown Source)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:361)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1081)
2015-05-25 21:39:11,028 [main-SendThread(192.168.140.142:2181)] INFO  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:975) - Opening socket connection to server 192.168.140.142/192.168.140.142:2181. Will not attempt to authenticate using SASL (unknown error)
2015-05-25 21:39:32,030 [main-SendThread(192.168.140.142:2181)] WARN  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:1102) - Session 0x0 for server null, unexpected error, closing socket connection and attempting reconnect
java.net.ConnectException: Connection timed out: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(Unknown Source)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:361)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1081)
2015-05-25 21:39:33,131 [main-SendThread(192.168.140.142:2181)] INFO  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:975) - Opening socket connection to server 192.168.140.142/192.168.140.142:2181. Will not attempt to authenticate using SASL (unknown error)
2015-05-25 21:39:54,133 [main-SendThread(192.168.140.142:2181)] WARN  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:1102) - Session 0x0 for server null, unexpected error, closing socket connection and attempting reconnect
java.net.ConnectException: Connection timed out: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(Unknown Source)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:361)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1081)
2015-05-25 21:39:55,235 [main-SendThread(192.168.140.142:2181)] INFO  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:975) - Opening socket connection to server 192.168.140.142/192.168.140.142:2181. Will not attempt to authenticate using SASL (unknown error)
2015-05-25 21:40:16,234 [main-SendThread(192.168.140.142:2181)] WARN  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:1102) - Session 0x0 for server null, unexpected error, closing socket connection and attempting reconnect
java.net.ConnectException: Connection timed out: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(Unknown Source)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:361)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1081)
2015-05-25 21:40:17,335 [main-SendThread(192.168.140.142:2181)] INFO  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:975) - Opening socket connection to server 192.168.140.142/192.168.140.142:2181. Will not attempt to authenticate using SASL (unknown error)
2015-05-25 21:40:38,333 [main-SendThread(192.168.140.142:2181)] WARN  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:1102) - Session 0x0 for server null, unexpected error, closing socket connection and attempting reconnect
java.net.ConnectException: Connection timed out: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(Unknown Source)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:361)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1081)
2015-05-25 21:40:39,434 [main-SendThread(192.168.140.142:2181)] INFO  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:975) - Opening socket connection to server 192.168.140.142/192.168.140.142:2181. Will not attempt to authenticate using SASL (unknown error)
2015-05-25 21:41:00,440 [main-SendThread(192.168.140.142:2181)] WARN  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:1102) - Session 0x0 for server null, unexpected error, closing socket connection and attempting reconnect
java.net.ConnectException: Connection timed out: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(Unknown Source)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:361)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1081)
2015-05-25 21:41:01,542 [main-SendThread(192.168.140.142:2181)] INFO  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:975) - Opening socket connection to server 192.168.140.142/192.168.140.142:2181. Will not attempt to authenticate using SASL (unknown error)
2015-05-25 21:41:22,546 [main-SendThread(192.168.140.142:2181)] WARN  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:1102) - Session 0x0 for server null, unexpected error, closing socket connection and attempting reconnect
java.net.ConnectException: Connection timed out: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(Unknown Source)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:361)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1081)
2015-05-25 21:41:23,647 [main-SendThread(192.168.140.142:2181)] INFO  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:975) - Opening socket connection to server 192.168.140.142/192.168.140.142:2181. Will not attempt to authenticate using SASL (unknown error)
2015-05-25 21:41:44,653 [main-SendThread(192.168.140.142:2181)] WARN  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:1102) - Session 0x0 for server null, unexpected error, closing socket connection and attempting reconnect
java.net.ConnectException: Connection timed out: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(Unknown Source)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:361)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1081)
2015-05-25 21:41:45,754 [main-SendThread(192.168.140.142:2181)] INFO  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:975) - Opening socket connection to server 192.168.140.142/192.168.140.142:2181. Will not attempt to authenticate using SASL (unknown error)
2015-05-25 21:42:06,752 [main-SendThread(192.168.140.142:2181)] WARN  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:1102) - Session 0x0 for server null, unexpected error, closing socket connection and attempting reconnect
java.net.ConnectException: Connection timed out: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(Unknown Source)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:361)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1081)
2015-05-25 21:42:07,853 [main-SendThread(192.168.140.142:2181)] INFO  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:975) - Opening socket connection to server 192.168.140.142/192.168.140.142:2181. Will not attempt to authenticate using SASL (unknown error)
2015-05-25 21:42:28,853 [main-SendThread(192.168.140.142:2181)] WARN  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:1102) - Session 0x0 for server null, unexpected error, closing socket connection and attempting reconnect
java.net.ConnectException: Connection timed out: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(Unknown Source)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:361)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1081)
2015-05-25 21:42:29,954 [main-SendThread(192.168.140.142:2181)] INFO  org.apache.zookeeper.ClientCnxn$SendThread (ClientCnxn.java:975) - Opening socket connection to server 192.168.140.142/192.168.140.142:2181. Will not attempt to authenticate using SASL (unknown error)
